<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet type="text/css" href="../../resources/stylesheets/wasmsgxml.css"?>
<TMSSource name="TivoliMessages" tmsVersion="1.0" xml:lang="en">
<!-- CMVC MSG File Name = ws/code/webcontainer/src/com/ibm/ws/webcontainer/resources/Messages.nlsprops -->
<!-- DO NOT EDIT THIS FILE - This file was generated by the XML/Html & Property emitter -->
<!-- BEGIN MESSAGES -->
<Message ID="SRVE8071E" severity="E" prefix="yes">
  <MsgText pgmKey="Null.webcontainer.config.object" varFormat="Java">
    Null.webcontainer.config.object=SRVE8071E: Objeto de configuración de contenedor web nulo.
  </MsgText>
  <Explanation>
    Se ha producido una excepción de configuración inesperada.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0109E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.Argument:.Invalid.ObjectPool.Instantiated." varFormat="Java">
    Illegal.Argument:.Invalid.ObjectPool.Instantiated.=SRVE0109E: Excepción de argumento no permitido: Instancia de ObjectPool no válida.
  </MsgText>
  <Explanation>
    Este es un error interno del sistema y está causado probablemente por una anomalía anterior.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema. Si no existen mensajes relacionados o no ayudan a resolver el problema, póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0221E" severity="E" prefix="yes">
  <MsgText pgmKey="Serving.JSP.Not.Allowed" varFormat="Java">
    Serving.JSP.Not.Allowed=SRVE0221E: El servicio de contenidos de archivos JSP no está permitido.
  </MsgText>
  <Explanation>
    No se pueden servir los contenidos en un archivo JSP
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8078E" severity="E" prefix="yes">
  <MsgText pgmKey="Null.servlet.config.object" varFormat="Java">
    Null.servlet.config.object=SRVE8078E: Objeto de configuración de servlet nulo.
  </MsgText>
  <Explanation>
    Se ha producido una excepción de configuración inesperada.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8101E" severity="E" prefix="yes">
  <MsgText pgmKey="Response.buffer.corruption.detected" varFormat="Java">
    Response.buffer.corruption.detected=SRVE8101E: Se ha detectado corrupción en el almacenamiento intermedio de respuesta. Se esperaba remaining={0}, actual={1}.
  </MsgText>
  <Explanation>
    Un error interno inesperado ha causado una no coincidencia entre el estado esperado del almacenamiento intermedio de salida y el estado actual.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0162I" severity="I" prefix="yes">
  <MsgText pgmKey="web.container.servlet.spec.level" varFormat="Java">
    web.container.servlet.spec.level=SRVE0162I: Nivel de especificación de servlet: 3.0
  </MsgText>
  <Explanation>
    El nivel de especificación de servlet soportado por el servidor de aplicaciones.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0245I" severity="I" prefix="yes">
  <MsgText pgmKey="reaper.removing.mapping.[{0}].for.servlet.[{1}]" varFormat="Java">
    reaper.removing.mapping.[{0}].for.servlet.[{1}]=SRVE0245I: La hebra segadora está eliminando la correlación: [{0}] para el servlet: [{0}].
  </MsgText>
  <Explanation>
    La hebra segadora elimina las correlaciones de JSPS y otras extensiones que han estado inactivos de forma que han superado el límite de inactividad de la segadora.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8060E" severity="E" prefix="yes">
  <MsgText pgmKey="exception.while.injecting.resource" varFormat="Java">
    exception.while.injecting.resource=SRVE8060E: Se ha producido una excepción inesperada durante la inyección de recursos.
  </MsgText>
  <Explanation>
    Se ha producido una excepción inesperada durante la inyección de recursos.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0138E" severity="E" prefix="yes">
  <MsgText pgmKey="postInvoke.Security.Exception." varFormat="Java">
    postInvoke.Security.Exception.=SRVE0138E: Excepción de seguridad postInvoke
  </MsgText>
  <Explanation>
    Se ha encontrado un problema de seguridad durante el proceso de la petición.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema.
  </UserResponse>
</Message>
<Message ID="SRVE0208E" severity="E" prefix="yes">
  <MsgText pgmKey="Unsupported.conversion" varFormat="Java">
    Unsupported.conversion=SRVE0208E: Conversión no soportada
  </MsgText>
  <Explanation>
    El código ha intentado hacer una conversión ilegal de un valor de cabecera, por ejemplo, desde int a Datos, cuando no se puede realizar dicha conversión. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Modifique el servlet/JSP que falla.
  </UserResponse>
</Message>
<Message ID="SRVE0287E" severity="E" prefix="yes">
  <MsgText pgmKey="server.root.is.null" varFormat="Java">
    server.root.is.null=SRVE0287E: server.root es nulo
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8113W" severity="W" prefix="yes">
  <MsgText pgmKey="AsyncContext.lock.already.held" varFormat="Java">
    AsyncContext.lock.already.held=SRVE8113W: No se puede obtener un bloqueo.  Otra hebra ya ha invocado el procesamiento de errores.
  </MsgText>
  <Explanation>
    Otra hebra ya ha invocado el procesamiento de errores.
  </Explanation>
  <UserResponse>
    Asegúrese de que la aplicación puede manejar una excepción aquí.  Si es necesario que se devuelva este método aunque no se realice la llamada, póngase en contacto con el soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0297E" severity="E" prefix="yes">
  <MsgText pgmKey="WebApp.destroy.encountered.errors" varFormat="Java">
    WebApp.destroy.encountered.errors=SRVE0297E: La destrucción de la aplicación web {0} se ha encontrado errores: {1}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0311E" severity="E" prefix="yes">
  <MsgText pgmKey="transport.error" varFormat="Java">
    transport.error=SRVE0311E: Se ha producido un error en el transporte: {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0016E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.Argument:.Invalid.Header.Format" varFormat="Java">
    Illegal.Argument:.Invalid.Header.Format=SRVE0016E: Excepción de argumento no permitido: Formato de cabecera no válido
  </MsgText>
  <Explanation>
    El formato encontrado para una cabecera de solicitud no es válido.
  </Explanation>
  <UserResponse>
    Compruebe la creación de cabecera del cliente.
  </UserResponse>
</Message>
<Message ID="SRVE8027E" severity="E" prefix="yes">
  <MsgText pgmKey="an.io.related.error.has.occurred.during.the.processing.of.the.given.AsyncEvent" varFormat="Java">
    an.io.related.error.has.occurred.during.the.processing.of.the.given.AsyncEvent=SRVE8027E: Se ha producido un error relacionado con la entrada/salida al procesar el AsyncEvent indicado.
  </MsgText>
  <Explanation>
    Los errores de E/S se pueden producir al invocar un AsyncListener si está escribiendo la respuesta simultáneamente junto con otra hebra o algo ha cerrado la respuesta.
  </Explanation>
  <UserResponse>
    Es recomendable que los AsyncListeners llamen únicamente a complete o dispatch, pero que no escriban en la respuesta. De lo contrario, el cliente será el responsable de efectuar su propia sincronización para evitar una IOException.
  </UserResponse>
</Message>
<Message ID="SRVE8018E" severity="E" prefix="yes">
  <MsgText pgmKey="error.validate.webServlet.zero.mappings" varFormat="Java">
    error.validate.webServlet.zero.mappings=SRVE8018E: La clase, {0}, no tenía patrones especificados en la anotación
  </MsgText>
  <Explanation>
    La clase debe especificar patrones en la anotación mediante el atributo de valor o urlPatterns
  </Explanation>
  <UserResponse>
    Modifique la clase añadir uno de los atributos de valor o urlPatterns
  </UserResponse>
</Message>
<Message ID="SRVE0292I" severity="I" prefix="yes">
  <MsgText pgmKey="log.servlet.message" varFormat="Java">
    log.servlet.message=SRVE0292I: Mensaje de servlet - [{0}]:.{1}
  </MsgText>
  <Explanation>
    Mensaje informativo anotado cronológicamente para un servlet mediante el uso de ServletContext.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0242I" severity="I" prefix="yes">
  <MsgText pgmKey="[{0}].Initialization.successful" varFormat="Java">
    [{0}].Initialization.successful=SRVE0242I: [{2}] [{1}] [{0}]: Inicialización satisfactoria.
  </MsgText>
  <Explanation>
    El método init() del recurso de destino se ha ejecutado satisfactoriamente y el destino se ha ubicado en el servicio.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8102E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.value.for.ResponseBuffer" varFormat="Java">
    Illegal.value.for.ResponseBuffer=SRVE8102E: Valor no permitido para la propiedad personalizada responsebuffersize [{0}].
  </MsgText>
  <Explanation>
    Valor no permitido para la propiedad personalizada responsebuffersize.
  </Explanation>
  <UserResponse>
    Compruebe el valor de la propiedad personalizada responsebuffersize.
  </UserResponse>
</Message>
<Message ID="SRVE0249W" severity="W" prefix="yes">
  <MsgText pgmKey="sync.tothread.not.set.for.server.[{0}]" varFormat="Java">
    sync.tothread.not.set.for.server.[{0}]=SRVE0249W: La aplicación {0} ha solicitado SyncToOSThread, pero el servidor no está habilitado para SyncToOSThread
  </MsgText>
  <Explanation>
    SyncToOSThread sincroniza la entidad de rol de Java EE con la hebra del sistema operativo, lo que quiere decir que la identidad de la hebra del sistema operativo se equipara con la identidad del rol de Java EE para la duración de la solicitud. Este mensaje indica que mientras se haya configurado una aplicación para que utilice la funcionalidad SyncToOSThread, el servidor en el que se instala no está configurado para soportar funciones de SyncToOSThread. Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Si se necesitan las funciones de SyncToOSThread para el servidor en cuestión, consulte el Information Center para obtener una descripción sobre cómo habilitar SyncToOSThread para un servidor.
  </UserResponse>
</Message>
<Message ID="SRVE8045E" severity="E" prefix="yes">
  <MsgText pgmKey="programmatic.sessions.disabled" varFormat="Java">
    programmatic.sessions.disabled=SRVE8045E: La configuración de la sesión relativa a la programación se ha inhabilitado en este servidor.
  </MsgText>
  <Explanation>
    La configuración de la sesión relativa a la programación se ha inhabilitado en este servidor.
  </Explanation>
  <UserResponse>
    Habilite la configuración de la sesión relativa a la programación.
  </UserResponse>
</Message>
<Message ID="SRVE0270E" severity="E" prefix="yes">
  <MsgText pgmKey="illegal.servlet.mapping" varFormat="Java">
    illegal.servlet.mapping=SRVE0270E: Se ha encontrado una correlación de servlets no permitida para el servlet {0}.
  </MsgText>
  <Explanation>
    La correlación de servlets especificada en webconfig.xml no es válida. Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Compruebe las correlaciones de servlets en webconfig.xml y realice los cambios adecuados
  </UserResponse>
</Message>
<Message ID="SRVE8061E" severity="E" prefix="yes">
  <MsgText pgmKey="unable.to.invoke.method" varFormat="Java">
    unable.to.invoke.method=SRVE8061E: No se ha podido invocar al método --&gt; [{0}] en la clase --&gt; [{1}].
  </MsgText>
  <Explanation>
    Se ha producido un problema al invocar postConstruct o preDestroy.
  </Explanation>
  <UserResponse>
    Consulte la causa raíz.
  </UserResponse>
</Message>
<Message ID="SRVE0133E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.Parsing.Parameters" varFormat="Java">
    Error.Parsing.Parameters=SRVE0133E: Se ha producido un error al analizar parámetros. {0}
  </MsgText>
  <Explanation>
    Problema encontrado al analizar los parámetros del servlet.
  </Explanation>
  <UserResponse>
    Verifique que el cliente esté enviando la solicitud completa.
  </UserResponse>
</Message>
<Message ID="SRVE0100E" severity="E" prefix="yes">
  <MsgText pgmKey="Uncaught.init().exception.thrown.by.servlet.{0}:.{2}" varFormat="Java">
    Uncaught.init().exception.thrown.by.servlet.{0}:.{2}=SRVE0100E: No se ha detectado la excepción init() creada por el servlet {0} en la aplicación {1}: {2}
  </MsgText>
  <Explanation>
    No se ha podido inicializar el servlet especificado. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Asegúrese de que el archivo de clases del servlet esté disponible en la vía de acceso de clases de la aplicación.
  </UserResponse>
</Message>
<Message ID="SRVE8105W" severity="W" prefix="yes">
  <MsgText pgmKey="unable.to.retrieve.timer.manager" varFormat="Java">
    unable.to.retrieve.timer.manager=SRVE8105W: Se ha producido un error al recuperar el gestor de temporizadores para iniciar tiempos de espera de servlets asíncronos. Se utilizará el temporizador de hebras predeterminado.
  </MsgText>
  <Explanation>
    La consola de administración no debería permitir la adopción de un nombre JNDI de gestor de temporizadores no válido.
  </Explanation>
  <UserResponse>
    Verifique que el gestor de temporizadores seleccionado en el panel del contenedor web sea válido.
  </UserResponse>
</Message>
<Message ID="SRVE8016E" severity="E" prefix="yes">
  <MsgText pgmKey="error.validate.annotation.extension.class" varFormat="Java">
    error.validate.annotation.extension.class=SRVE8016E: La anotación {0} no se puede aplicar a la clase, {1}, porque amplía la superclase incorrecta
  </MsgText>
  <Explanation>
    Algunas anotaciones sólo se pueden aplicar a clases que amplían el tipo correcto.
  </Explanation>
  <UserResponse>
    Modifique la clase para que amplíe la superclase correcta
  </UserResponse>
</Message>
<Message ID="SRVE8010E" severity="E" prefix="yes">
  <MsgText pgmKey="request.does.not.support.async.servlet.processing" varFormat="Java">
    request.does.not.support.async.servlet.processing=SRVE8010E: LA solicitud actual no admite el proceso de servlets asíncrono.
  </MsgText>
  <Explanation>
    Uno de los servlets o filtros de la solicitud no soporta los servlets asíncronos.
  </Explanation>
  <UserResponse>
    No utilice servlets asíncronos ni habilite el proceso asíncrono para todos los servlets y filtros de la cadena añadiendo el elemento soportado asíncrono. a las definiciones de componente.
  </UserResponse>
</Message>
<Message ID="SRVE0254E" severity="E" prefix="yes">
  <MsgText pgmKey="unsupported.request.encoding.[{0}]" varFormat="Java">
    unsupported.request.encoding.[{0}]=SRVE0254E: No se ha podido establecer la codificación de caracteres de la solicitud: [{0}].
  </MsgText>
  <Explanation>
    La codificación de la solicitud especificada en la solicitud no es válida. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Revise los registros para determinar si la codificación especificada es una codificación válida.
  </UserResponse>
</Message>
<Message ID="SRVE8091W" severity="W" prefix="yes">
  <MsgText pgmKey="servlet.wrapper.unavailable" varFormat="Java">
    servlet.wrapper.unavailable=SRVE8091W: El servlet {0} actualmente no está disponible: {1}
  </MsgText>
  <Explanation>
    Se ha emitido una UnavailableException desde el servlet.
  </Explanation>
  <UserResponse>
    Consulte la causa raíz.
  </UserResponse>
</Message>
<Message ID="SRVE0320E" severity="E" prefix="yes">
  <MsgText pgmKey="Filter.found.but.injection.failure" varFormat="Java">
    Filter.found.but.injection.failure=SRVE0320E: Se ha encontrado el filtro [{0}], pero se ha producido una anomalía en la inyección de recursos.
  </MsgText>
  <Explanation>
    El filtro no ha podido inicializarse debido a que se ha producido un error cuando se inyectaba un recurso.
  </Explanation>
  <UserResponse>
    Verifique que existan los recursos inyectados y que sean accesibles para el filtro.
  </UserResponse>
</Message>
<Message ID="SRVE0315E" severity="E" prefix="yes">
  <MsgText pgmKey="Exception" varFormat="Java">
    Exception=SRVE0315E: Se ha producido una excepción: {0}
  </MsgText>
  <Explanation>
    Se ha producido una excepción.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0060E" severity="E" prefix="yes">
  <MsgText pgmKey="Unable.to.bind.hostname.[{0}].to.servletHost.[{1}]" varFormat="Java">
    Unable.to.bind.hostname.[{0}].to.servletHost.[{1}]=SRVE0060E: No se ha podido enlazar el nombre de host [{0}] con servletHost [{1}]
  </MsgText>
  <Explanation>
    No se ha podido enlazar el nombre de host especificado con el host de servlet especificado.
  </Explanation>
  <UserResponse>
    Asegúrese de que el nombre del host se ha especificado correctamente y de que el host de servlet se ha especificado en virtualhosts.xml.
  </UserResponse>
</Message>
<Message ID="SRVE0103E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.Argument:.ScriptName.must.be.the.first.part.of.the.URI" varFormat="Java">
    Illegal.Argument:.ScriptName.must.be.the.first.part.of.the.URI=SRVE0103E: Excepción de argumento no permitido: ScriptName debe ser la primera parte del URI
  </MsgText>
  <Explanation>
    Este es un error interno del sistema y está causado probablemente por una anomalía anterior.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema. Si no existen mensajes relacionados o no ayudan a resolver el problema, póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0308E" severity="E" prefix="yes">
  <MsgText pgmKey="filtering.by.asterisk" varFormat="Java">
    filtering.by.asterisk=SRVE0308E: No se permite el filtrado por asterisco.
  </MsgText>
  <Explanation>
    No se permite realizar filtrados por asterisco porque la inhabilitación del servicio a todos los servlets por nombre de clase está manejada por la propiedad personalizada com.ibm.ws.webcontainer.disallowserveservletsbyclassname. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Establezca com.ibm.ws.webcontainer.disallowserveservletsbyclassname en verdadero para bloquear el servicio a todas las clases por nombre de clase.
  </UserResponse>
</Message>
<Message ID="SRVE0294E" severity="E" prefix="yes">
  <MsgText pgmKey="log.servlet.error.and.message" varFormat="Java">
    log.servlet.error.and.message=SRVE0294E: [Error de servlet]-[{0}]: {1}: {2}
  </MsgText>
  <Explanation>
    Mensaje de error y excepción anotados cronológicamente para un servlet mediante el uso de ServletContext.
  </Explanation>
  <UserResponse>
    Revise el mensaje y la excepción para determinar la causa del error y la acción necesaria. Puede haber información adicional en las anotaciones cronológicas del servidor.
  </UserResponse>
</Message>
<Message ID="SRVE0268E" severity="E" prefix="yes">
  <MsgText pgmKey="error.obtaining.session.context" varFormat="Java">
    error.obtaining.session.context=SRVE0268E: Se ha producido un error al obtener el contexto de sesión para la aplicación web: {0}
  </MsgText>
  <Explanation>
    No se ha podido obtener un contexto de sesión válido para la aplicación web. Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Compruebe que el gestor de sesiones esté bien configurado.
  </UserResponse>
</Message>
<Message ID="SRVE0215E" severity="E" prefix="yes">
  <MsgText pgmKey="non-HTTP.request.or.response" varFormat="Java">
    non-HTTP.request.or.response=SRVE0215E: petición o respuesta no HTTP
  </MsgText>
  <Explanation>
    La cadena de servlets que se ha definido sólo puede manejar solicitudes HTTP. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8054E" severity="E" prefix="yes">
  <MsgText pgmKey="setting.maxIdleTime.to.default.value.of.60000" varFormat="Java">
    setting.maxIdleTime.to.default.value.of.60000=SRVE8054E: Se ha producido una NumberFormatException al intentar establecer el tiempo de inactividad máximo del servlet de modelo de hebra único.
  </MsgText>
  <Explanation>
    Se ha producido una NumberFormatException al intentar establecer el tiempo de inactividad máximo del servlet de modelo de hebra único.
  </Explanation>
  <UserResponse>
    Verifique que el valor del tiempo de inactividad máximo del servlet del modelo de hebra único sea un entero válido.
  </UserResponse>
</Message>
<Message ID="SRVE0307W" severity="W" prefix="yes">
  <MsgText pgmKey="Null.Filter.Mapping" varFormat="Java">
    Null.Filter.Mapping=SRVE0307W: Correlación nula para el filtro. Esto puede ocurrir si intenta especificar una correlación * en una aplicación anterior a 2.5
  </MsgText>
  <Explanation>
    No había ninguna correlación de servlets para el filtro por lo que no se ha podido añadir. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Cambie los descriptores de la aplicación al nivel 2.5 si está utilizando una correlación *.
  </UserResponse>
</Message>
<Message ID="SRVE0304E" severity="E" prefix="yes">
  <MsgText pgmKey="servletrequestwrapper.is.not.an.instance.of.so.wrapped.logic.may.be.bypassed" varFormat="Java">
    servletrequestwrapper.is.not.an.instance.of.so.wrapped.logic.may.be.bypassed=SRVE0304E: ServletRequestWrapper [{0}] no es una instancia de [{1}] por lo que es posible que se eluda la lógica derivada.
  </MsgText>
  <Explanation>
    El contenedor web espera que un objeto de petición de acomodación sea una instancia de una clase de la que no lo es. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Cambie el código para implementar la interfaz esperada.
  </UserResponse>
</Message>
<Message ID="SRVE0086E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.Argument:.Missing.Resource.Bootstrap.Properties" varFormat="Java">
    Illegal.Argument:.Missing.Resource.Bootstrap.Properties=SRVE0086E: Excepción de argumento no permitido: Faltan propiedades de programa de arranque de recurso
  </MsgText>
  <Explanation>
    Error interno del servidor.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0288E" severity="E" prefix="yes">
  <MsgText pgmKey="chown.failed" varFormat="Java">
    chown.failed=SRVE0288E: chown (CHange OWNer) fallido; se ha intentado ejecutar el mandato: [{0}] código de salida: [{1}]
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8069E" severity="E" prefix="yes">
  <MsgText pgmKey="null.session.registry" varFormat="Java">
    null.session.registry=SRVE8069E: El registro de sesión es NULL.
  </MsgText>
  <Explanation>
    El Registro de sesión es NULL posiblemente debido a que SessionManager no se ha iniciado correctamente. Las aplicaciones web FALLARÁN.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0246I" severity="I" prefix="yes">
  <MsgText pgmKey="reaper.destroying.servlet.[{0}]" varFormat="Java">
    reaper.destroying.servlet.[{0}]=SRVE0246I: La hebra segadora esta destruyendo el servlet: [{0}].
  </MsgText>
  <Explanation>
    Ninguna.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0194E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.from.included.servlet" varFormat="Java">
    Illegal.from.included.servlet=SRVE0194E: Argumento no permitido del servlet incluido
  </MsgText>
  <Explanation>
    La operación intentada no se puede realizar en un servlet/JSP incluido (consulte la Especificación del servlet). Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Modifique el servlet/JSP que falla.
  </UserResponse>
</Message>
<Message ID="SRVE0207E" severity="E" prefix="yes">
  <MsgText pgmKey="Uncaught.initialization.exception.thrown.by.servlet" varFormat="Java">
    Uncaught.initialization.exception.thrown.by.servlet=SRVE0207E: No se ha detectado la excepción de inicialización creada por el servlet
  </MsgText>
  <Explanation>
    El destino ha emitido una excepción durante una inicialización que no ha sido detectada por el código de usuario. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    La excepción se justifica en el código de usuario.
  </UserResponse>
</Message>
<Message ID="SRVE8023E" severity="E" prefix="yes">
  <MsgText pgmKey="multipart.file.upload.exception" varFormat="Java">
    multipart.file.upload.exception=SRVE8023E: Se ha producido una excepción al cargar el archivo.
  </MsgText>
  <Explanation>
    Se ha producido una excepción al cargar el archivo.
  </Explanation>
  <UserResponse>
    Investigue el error anidado y realice la acción adecuada.
  </UserResponse>
</Message>
<Message ID="SRVE8009E" severity="E" prefix="yes">
  <MsgText pgmKey="error.while.initializing.target.mappings" varFormat="Java">
    error.while.initializing.target.mappings=SRVE8009E: Se ha producido un error de servidor interno inesperado al inicializar las correlaciones de destino.
  </MsgText>
  <Explanation>
    Se ha producido un error de servidor interno inesperado al inicializar las correlaciones de destino.
  </Explanation>
  <UserResponse>
    Recopile los registros de servidor y póngase en contacto con el soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8006E" severity="E" prefix="yes">
  <MsgText pgmKey="add.filter.mapping.to.empty.url.patterns" varFormat="Java">
    add.filter.mapping.to.empty.url.patterns=SRVE8006E: Se ha intentado añadir una correlación de filtros con un parámetro de patrón URL de argumento variable vacío.
  </MsgText>
  <Explanation>
    El parámetro de patrón URL del argumento variable pasado a FilterRegistration.addMappingForUrlPatterns no debe ser nulo ni debe estar vacío.
  </Explanation>
  <UserResponse>
    Modifique el código de aplicación de acuerdo con la explicación del error.
  </UserResponse>
</Message>
<Message ID="SRVE0264E" severity="E" prefix="yes">
  <MsgText pgmKey="error.initializing.extension.factories" varFormat="Java">
    error.initializing.extension.factories=SRVE0264E: Se ha producido un error al inicializar Extension Factories: {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0228E" severity="E" prefix="yes">
  <MsgText pgmKey="classname.defined.in.server.using.proper.case.and.fully.qualified.package" varFormat="Java">
    classname.defined.in.server.using.proper.case.and.fully.qualified.package=SRVE0228E: Compruebe que el nombre de la clase se haya definido en el servidor con las mayúsculas y minúsculas como corresponda y con el paquete plenamente cualificado.
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Compruebe que el nombre de la clase se haya definido en el servidor con las mayúsculas y minúsculas como corresponda y con el paquete plenamente cualificado.
  </UserResponse>
</Message>
<Message ID="SRVE0310E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.in.WebContainer.custom.property.value" varFormat="Java">
    Error.in.WebContainer.custom.property.value=SRVE0310E: Se ha producido un error en una propiedad personalizada: {1}
  </MsgText>
  <Explanation>
    Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Verifique que la propiedad personalizada sea correcta
  </UserResponse>
</Message>
<Message ID="SRVE0120E" severity="E" prefix="yes">
  <MsgText pgmKey="IO.Error" varFormat="Java">
    IO.Error=SRVE0120E: Error de entrada/salida {0}
  </MsgText>
  <Explanation>
    Se ha encontrado un error de ES.
  </Explanation>
  <UserResponse>
    Varía en función de la causa raíz.
  </UserResponse>
</Message>
<Message ID="SRVE8012E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.adding.ServletContextListener" varFormat="Java">
    Error.adding.ServletContextListener=SRVE8012E: Se ha producido un error al añadir ServletContextListener
  </MsgText>
  <Explanation>
    Un ServletContextListener no se puede añadir desde otro ServletContextListener
  </Explanation>
  <UserResponse>
    Añada el ServletContextListener a web.xml o a web-fragment.xml, anótelo con la anotación @WebListener o añádalo mediante programación desdeServletContainerInitializer#onStartup
  </UserResponse>
</Message>
<Message ID="SRVE8022E" severity="E" prefix="yes">
  <MsgText pgmKey="multipart.request.size.too.big" varFormat="Java">
    multipart.request.size.too.big=SRVE8022E: La solicitud es demasiado grande.
  </MsgText>
  <Explanation>
    La solicitud en varias partes/de datos de formulario supera el tamaño máximo configurado.
  </Explanation>
  <UserResponse>
    Aumente el atributo de tamaño de la solicitud de la configuración en varias partes del servlet indicado.
  </UserResponse>
</Message>
<Message ID="SRVE0213E" severity="E" prefix="yes">
  <MsgText pgmKey="class.not.found" varFormat="Java">
    class.not.found=SRVE0213E: clase no encontrada
  </MsgText>
  <Explanation>
    Excepción interna.
  </Explanation>
  <UserResponse>
    Si el problema persiste, puede haber información adicional disponible si busca el ID del mensaje en los sitios Web siguientes: página de soporte de IBM WebSphere Application Server: http://www.ibm.com/software/webservers/appserv/was/support/   página de soporte de IBM WebSphere Application Server for z/OS: http://www.ibm.com/software/webservers/appserv/zos_os390/support/
  </UserResponse>
</Message>
<Message ID="SRVE8096W" severity="W" prefix="yes">
  <MsgText pgmKey="exception.scheduling.work.item" varFormat="Java">
    exception.scheduling.work.item=SRVE8096W: Se ha producido una excepción al planificar el ejecutable [{0}].
  </MsgText>
  <Explanation>
    Al planificar un elemento de trabajo con un gestor de trabajo de bean asíncrono, es posible que se emita una excepción si la cola de trabajos está llena y va a fallar la acción de la cola de trabajos completa que se ha configurado.
  </Explanation>
  <UserResponse>
    Evalúe si la acción de la cola de trabajos completa debería bloquearse hasta que la cola no estuviera llena o procese esta excepción en su código de aplicación.
  </UserResponse>
</Message>
<Message ID="SRVE8011E" severity="E" prefix="yes">
  <MsgText pgmKey="Unsupported.op.from.servlet.context.listener" varFormat="Java">
    Unsupported.op.from.servlet.context.listener=SRVE8011E: Esta operación no se puede ejecutar desde un escucha añadido mediante programación. (Operación: {0} | Escucha: {1} | Aplicación: {2})
  </MsgText>
  <Explanation>
    Algunas operaciones no se pueden ejecutar desde una escucha añadida mediante programación puesto que no hay ninguna opción de configuración que restrinja su ejecución cuando se definen en un Descriptor de bibliotecas de etiquetas (Tag Library Descriptor - TLD).
  </Explanation>
  <UserResponse>
    Añada el escucha mediante una anotación o mediante el web.xml para permitir la ejecución de este método.
  </UserResponse>
</Message>
<Message ID="SRVE8082E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.while.adding.static.file.processor" varFormat="Java">
    Error.while.adding.static.file.processor=SRVE8082E: Se ha producido un error de servidor interno inesperado al añadir el procesador de archivos estático.
  </MsgText>
  <Explanation>
    Se ha producido un error de servidor interno inesperado al añadir el procesador de archivos estático.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8021E" severity="E" prefix="yes">
  <MsgText pgmKey="multipart.file.size.too.big" varFormat="Java">
    multipart.file.size.too.big=SRVE8021E: El archivo que se está cargando es demasiado grande.
  </MsgText>
  <Explanation>
    El archivo que se está cargando mediante la solicitud en varias partes/de datos de formulario supera el tamaño máximo configurado.
  </Explanation>
  <UserResponse>
    Aumente el atributo de tamaño de archivo de la configuración en varias partes del servlet indicado.
  </UserResponse>
</Message>
<Message ID="SRVE0250I" severity="I" prefix="yes">
  <MsgText pgmKey="module.[{0}].successfully.bound.to.virtualhost.[{1}]" varFormat="Java">
    module.[{0}].successfully.bound.to.virtualhost.[{1}]=SRVE0250I: El módulo web {0} se ha enlazado con {1}.
  </MsgText>
  <Explanation>
    El módulo web especificado se ha asociado con el host virtual mencionado y se puede acceder al mismo mediante cualquiera de las combinaciones de host/puerto de dicho host virtual.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0253I" severity="I" prefix="yes">
  <MsgText pgmKey="[{0}].Destroy.successful" varFormat="Java">
    [{0}].Destroy.successful=SRVE0253I: [{2}] [{1}] [{0}]: Destrucción satisfactoria.
  </MsgText>
  <Explanation>
    El método destroy() del recurso de destino se ha ejecutado satisfactoriamente y el destino se ha eliminado del servicio.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8001E" severity="E" prefix="yes">
  <MsgText pgmKey="error.adding.servlet.after.initialization" varFormat="Java">
    error.adding.servlet.after.initialization=SRVE8001E: Se ha intentado añadir un servlet dinámicamente tras la inicialización de ServletContext.
  </MsgText>
  <Explanation>
    Los servlets sólo se pueden añadir dinámicamente antes de inicializar ServletContext.
  </Explanation>
  <UserResponse>
    Mueva el código llamando al método actual en un método ServletContainerInitalizer&apos;s onStartup o ServletContextListener&apos;s contextInitialized.
  </UserResponse>
</Message>
<Message ID="SRVE0042E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.Argument.Invalid.Content.Length" varFormat="Java">
    Illegal.Argument.Invalid.Content.Length=SRVE0042E: Excepción de argumento no permitido: Longitud de contenido no válida
  </MsgText>
  <Explanation>
    Este es un error interno del sistema y está causado probablemente por una anomalía anterior. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema. Si no existen mensajes relacionados o no ayudan a resolver el problema, póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0139E" severity="E" prefix="yes">
  <MsgText pgmKey="preInvoke.Security.Exception." varFormat="Java">
    preInvoke.Security.Exception.=SRVE0139E: Excepción en la seguridad preInvoke {0}
  </MsgText>
  <Explanation>
    Problema de seguridad encontrado durante el proceso de la petición. Es un error del sistema.
  </Explanation>
  <UserResponse>
    Varía en función de la causa raíz.
  </UserResponse>
</Message>
<Message ID="SRVE0293E" severity="E" prefix="yes">
  <MsgText pgmKey="log.servlet.error" varFormat="Java">
    log.servlet.error=SRVE0293E: [Error de servlet]-[{0}]: {1}
  </MsgText>
  <Explanation>
    Excepción de error anotada cronológicamente para un servlet mediante el uso de ServletContext.
  </Explanation>
  <UserResponse>
    Revise la excepción para determinar la causa del error y la acción necesaria. Puede haber información adicional en las anotaciones cronológicas del servidor.
  </UserResponse>
</Message>
<Message ID="SRVE0275E" severity="E" prefix="yes">
  <MsgText pgmKey="error.adding.servlet.mapping.for.servlet" varFormat="Java">
    error.adding.servlet.mapping.for.servlet=SRVE0275E: Se ha producido un error al añadir la correlación de servlets para el [{0}] en la aplicación [{1}]: {2}
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Verifique que la correlación sea válida.
  </UserResponse>
</Message>
<Message ID="SRVE0280E" severity="E" prefix="yes">
  <MsgText pgmKey="extension.processor.failed.to.initialize.in.factory" varFormat="Java">
    extension.processor.failed.to.initialize.in.factory=SRVE0280E: El procesador de extensiones no se ha podido inicializar en la fábrica [{0}]: {1}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE8049E" severity="E" prefix="yes">
  <MsgText pgmKey="Exception.occured.during.servlet.destroy" varFormat="Java">
    Exception.occured.during.servlet.destroy=SRVE8049E: Se ha producido un error de servidor interno al destruir el servlet.
  </MsgText>
  <Explanation>
    Se ha producido un error de servidor interno al destruir el servlet.
  </Explanation>
  <UserResponse>
    Consulte la causa raíz de la excepción.
  </UserResponse>
</Message>
<Message ID="SRVE0243I" severity="I" prefix="yes">
  <MsgText pgmKey="reaper.with.interval.[{0}].limit.[{1}].started" varFormat="Java">
    reaper.with.interval.[{0}].limit.[{1}].started=SRVE0243I: Se ha iniciado el intervalo de hebras reaper: [{0}] y límite inactivo: [{1}].
  </MsgText>
  <Explanation>
    Ninguna.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8002E" severity="E" prefix="yes">
  <MsgText pgmKey="error.adding.filter.after.initialization" varFormat="Java">
    error.adding.filter.after.initialization=SRVE8002E: Se ha intentado añadir un filtro dinámicamente tras la inicialización de ServletContext.
  </MsgText>
  <Explanation>
    Los filtros sólo se pueden añadir dinámicamente antes de inicializar ServletContext.
  </Explanation>
  <UserResponse>
    Mueva el código llamando al método actual en un método ServletContainerInitalizer&apos;s onStartup o ServletContextListener&apos;s contextInitialized.
  </UserResponse>
</Message>
<Message ID="SRVE0181I" severity="I" prefix="yes">
  <MsgText pgmKey="[{0}][{1}][Servlet.LOG]:.{2}:.{3}" varFormat="Java">
    [{0}][{1}][Servlet.LOG]:.{2}:.{3}=SRVE0181I: [{0}] [{1}] [Servlet.LOG]: {2}: {3}
  </MsgText>
  <Explanation>
    Mensaje informativo y rastreo de pila anotado cronológicamente a través de ServletContext.log desde una aplicación.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8084E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.occurred.while.populating.the.namespace" varFormat="Java">
    Error.occurred.while.populating.the.namespace=SRVE8084E: Se ha producido un error de servidor interno inesperado al llenar el espacio de nombres. Nombre de la aplicación: {0}, Excepción : {1}.
  </MsgText>
  <Explanation>
    Se ha producido un error inesperado al llenar el espacio de nombres.
  </Explanation>
  <UserResponse>
    Compruebe los registros y la excepción del servidor para determinar qué podría haber causado el problema de enlace de espacio de nombres.
  </UserResponse>
</Message>
<Message ID="SRVE0031E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.Argument:.{0}.is.not.a.directory." varFormat="Java">
    Illegal.Argument:.{0}.is.not.a.directory.=SRVE0031E: Excepción de argumento no permitido: {0} no es un directorio.
  </MsgText>
  <Explanation>
    La vía de acceso especificada no es un directorio.
  </Explanation>
  <UserResponse>
    Asegúrese de que el directorio existe o que está solicitando el directorio correcto.
  </UserResponse>
</Message>
<Message ID="SRVE8075E" severity="E" prefix="yes">
  <MsgText pgmKey="Null.cachedServlet.object" varFormat="Java">
    Null.cachedServlet.object=SRVE8075E: Objeto de servlet de memoria caché nulo.
  </MsgText>
  <Explanation>
    Se ha producido una excepción de configuración inesperada.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8040E" severity="E" prefix="yes">
  <MsgText pgmKey="cannot.call.startAsync.multiple.times.within.same.dispatch" varFormat="Java">
    cannot.call.startAsync.multiple.times.within.same.dispatch=SRVE8040E: No se puede llamar startAsync varias veces dentro del mismo dispatch.
  </MsgText>
  <Explanation>
    El método startAsync no se puede llamar varias veces desde dentro del mismo dispatch. Una llamada a AsyncContext.dispatch le permitiría volver a llamar la startAsync.
  </Explanation>
  <UserResponse>
    Verifique que no haya problemas de lógica de código o de temporización que causen este comportamiento no permitido.
  </UserResponse>
</Message>
<Message ID="SRVE0247E" severity="E" prefix="yes">
  <MsgText pgmKey="reaper.thread.error" varFormat="Java">
    reaper.thread.error=SRVE0247E: Se ha producido un error al ejecutar la hebra segadora.
  </MsgText>
  <Explanation>
    Se ha producido una excepción durante el proceso de la hebra segadora. Es un error del sistema.
  </Explanation>
  <UserResponse>
    El proceso de segado es una forma de recogida de basura.
  </UserResponse>
</Message>
<Message ID="SRVE0301I" severity="I" prefix="yes">
  <MsgText pgmKey="encountered.problems.while.removing.servlet" varFormat="Java">
    encountered.problems.while.removing.servlet=SRVE0301I: Se han encontrado problemas al eliminar el servlet {0}. Detalles: {1}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE8029E" severity="E" prefix="yes">
  <MsgText pgmKey="exception.creating.search.path" varFormat="Java">
    exception.creating.search.path=SRVE8029E: Se ha producido una IOException al crear la vía de acceso de búsqueda directorio raíz de documentos ampliado.
  </MsgText>
  <Explanation>
    Puesto que la creación de la vía de acceso de búsqueda implica acceder al sistema de archivos, es posible que se produzca IOExceptions.
  </Explanation>
  <UserResponse>
    Verifique que la vía de acceso sea correcta y que haya problemas de hardware.
  </UserResponse>
</Message>
<Message ID="SRVE0303E" severity="E" prefix="yes">
  <MsgText pgmKey="servlet.name.for.servlet.mapping.not.found" varFormat="Java">
    servlet.name.for.servlet.mapping.not.found=SRVE0303E: No se ha podido encontrar el nombre del servlet de la correlación de servlets {0}.
  </MsgText>
  <Explanation>
    No hay ninguna definición de servlet para la correlación de servlets.
  </Explanation>
  <UserResponse>
    Compruebe que su web.xml tenga una definición de servlet para esta correlación de servlets.
  </UserResponse>
</Message>
<Message ID="SRVE0276E" severity="E" prefix="yes">
  <MsgText pgmKey="error.initializing.servlet" varFormat="Java">
    error.initializing.servlet=SRVE0276E: Se ha producido un error al inicializar el servlet [{0}]: {1}
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE8072E" severity="E" prefix="yes">
  <MsgText pgmKey="Null.virtual.host.config.object" varFormat="Java">
    Null.virtual.host.config.object=SRVE8072E: Objeto de configuración de host virtual nulo.
  </MsgText>
  <Explanation>
    Se ha producido una excepción de configuración inesperada.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0115E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.occurred.while.invoking.error.reporter" varFormat="Java">
    Error.occurred.while.invoking.error.reporter=SRVE0115E: Se ha producido un error al invocar al informador de errores {0}
  </MsgText>
  <Explanation>
    Ha habido un problema al invocar el informador de errores especificado.
  </Explanation>
  <UserResponse>
    Si el problema persiste, consulte la información de determinación de problemas en la página de soporte de WebSphere Application Server en http://www.ibm.com/software/webservers/appserv/was/support/.
  </UserResponse>
</Message>
<Message ID="SRVE8053E" severity="E" prefix="yes">
  <MsgText pgmKey="setting.initialPoolSize.to.default.value.of.5" varFormat="Java">
    setting.initialPoolSize.to.default.value.of.5=SRVE8053E: Se ha producido una NumberFormatException al intentar establecer el tamaño de agrupación de modelo de hebra único.
  </MsgText>
  <Explanation>
    Se ha producido una NumberFormatException al intentar establecer el tamaño de agrupación de modelo de hebra único.
  </Explanation>
  <UserResponse>
    Verifique que el valor del tamaño de agrupación de modelos de hebra único sea un entero válido.
  </UserResponse>
</Message>
<Message ID="SRVE8024E" severity="E" prefix="yes">
  <MsgText pgmKey="multipart.request.not.multipart" varFormat="Java">
    multipart.request.not.multipart=SRVE8024E: La solicitud no es del tipo en varias partes/de datos de formulario.
  </MsgText>
  <Explanation>
    La solicitud debe ser del tipo en varias partes/de datos de formulario para poder manejar este método.
  </Explanation>
  <UserResponse>
    Modifique los servlets realizando solicitudes previas tales como que la solicitud actual sea del tipo en varias partes/de datos de formulario.
  </UserResponse>
</Message>
<Message ID="SRVE0227E" severity="E" prefix="yes">
  <MsgText pgmKey="class.resides.in.proper.package.directory" varFormat="Java">
    class.resides.in.proper.package.directory=SRVE0227E: Compruebe que la clase se encuentre en el directorio del paquete correcto.
  </MsgText>
  <Explanation>
    El nombre de paquete del recurso no coincide con el paquete de archivos de clase compilado. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Compruebe que la clase se encuentre en el directorio del paquete correcto.
  </UserResponse>
</Message>
<Message ID="SRVE0209E" severity="E" prefix="yes">
  <MsgText pgmKey="Writer.already.obtained" varFormat="Java">
    Writer.already.obtained=SRVE0209E: YA se ha obtenido el grabador
  </MsgText>
  <Explanation>
    Ya se ha obtenido el grabador de esta respuesta. Utilice el grabador existente. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Revise la aplicación para determinar si se ha obtenido tanto PrintWriter como OuputStream. Sólo modifique el servlet/JSP que falla para obtener el uno o el otro.
  </UserResponse>
</Message>
<Message ID="SRVE0080E" severity="E" prefix="yes">
  <MsgText pgmKey="Invalid.Content.Length" varFormat="Java">
    Invalid.Content.Length=SRVE0080E: Longitud de contenido no válida
  </MsgText>
  <Explanation>
    La longitud del contenido no coincide con la longitud de los datos publicados. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Realice una investigación en el cliente para determinar porqué no coincide el campo de longitud de contenido con el tamaño de los datos publicados. Si la configuración del cliente es correcta, investigue en qué punto de la red el campo de longitud de contenido no es válido.
  </UserResponse>
</Message>
<Message ID="SRVE0299E" severity="E" prefix="yes">
  <MsgText pgmKey="could.not.create.wrapper.for.servlet" varFormat="Java">
    could.not.create.wrapper.for.servlet=SRVE0299E: No se ha podido crear un derivador para el servlet [{0}]
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0302E" severity="E" prefix="yes">
  <MsgText pgmKey="error.while.adding.static.file.processor" varFormat="Java">
    error.while.adding.static.file.processor=SRVE0302E: Se ha producido un error al añadir el procesador de archivos estático: {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE8038E" severity="E" prefix="yes">
  <MsgText pgmKey="trying.to.do.startAsync.after.a.complete" varFormat="Java">
    trying.to.do.startAsync.after.a.complete=SRVE8038E: Se está intentando realizar startAsync después de una compleción.
  </MsgText>
  <Explanation>
    El método startAsync no se puede llamar después de una compleción por la Especificación del servlet.
  </Explanation>
  <UserResponse>
    Verifique que no haya problemas de lógica de código o de temporización que causen este comportamiento no permitido.
  </UserResponse>
</Message>
<Message ID="SRVE0259W" severity="W" prefix="yes">
  <MsgText pgmKey="property.configuration.change" varFormat="Java">
    property.configuration.change=SRVE0259W: La propiedad {0} ahora sólo se puede configurar a nivel de contenedor web.
  </MsgText>
  <Explanation>
    La propiedad no se puede configurar a nivel de transporte y debe configurarse a nivel de contenedor web. Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Configure esta propiedad en el nivel del contenedor web.
  </UserResponse>
</Message>
<Message ID="SRVE0217E" severity="E" prefix="yes">
  <MsgText pgmKey="{0}.is.not.a.valid.class" varFormat="Java">
    {0}.is.not.a.valid.class=SRVE0217E: {0} no es una clase válida
  </MsgText>
  <Explanation>
    Este error se ha producido cuando el contenedor web ha intentado cargar una clase de implementación interna. Es un error del sistema.
  </Explanation>
  <UserResponse>
    Asegúrese de que la vía de acceso de clases del producto original no se haya modificado.
  </UserResponse>
</Message>
<Message ID="SRVE8080E" severity="E" prefix="yes">
  <MsgText pgmKey="isUserInRole.caught.exception" varFormat="Java">
    isUserInRole.caught.exception=SRVE8080E: Excepción isUserInRole inesperada, devolución de isUserInRole falsa.
  </MsgText>
  <Explanation>
    Excepción isUserInRole inesperada, devolución de isUserInRole falsa.
  </Explanation>
  <UserResponse>
    Excepción isUserInRole inesperada, devolución de isUserInRole falsa.
  </UserResponse>
</Message>
<Message ID="SRVE0126E" severity="E" prefix="yes">
  <MsgText pgmKey="Invalidation.Exception:.{0}.was.created" varFormat="Java">
    Invalidation.Exception:.{0}.was.created=SRVE0126E: Excepción de invalidación: se ha creado {0}
  </MsgText>
  <Explanation>
    Este es un error interno del sistema y está causado probablemente por una anomalía anterior.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema. Si no existen mensajes relacionados o no ayudan a resolver el problema, póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8068E" severity="E" prefix="yes">
  <MsgText pgmKey="Could.not.remove.webmodule" varFormat="Java">
    Could.not.remove.webmodule=SRVE8068E: No se ha podido eliminar el módulo web [{0}]
  </MsgText>
  <Explanation>
    Se ha producido una excepción inesperada al eliminar el módulo web.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8032W" severity="W" prefix="yes">
  <MsgText pgmKey="exception.occurred.while.initializing.ServletContainerInitializers.HandlesTypes" varFormat="Java">
    exception.occurred.while.initializing.ServletContainerInitializers.HandlesTypes=SRVE8032W: Se ha producido un problema al buscar una clase HandlesTypes durante la exploración de ServletContainerInitializer, {0}, para {1}.
  </MsgText>
  <Explanation>
    La clase no se ha encontrado en el cargador de clases al procesar la anotación ServletContainerInitializer HandlesTypes.
  </Explanation>
  <UserResponse>
    Asegúrese de que exista la clase dentro del cargador de clases del módulo web.
  </UserResponse>
</Message>
<Message ID="SRVE8046E" severity="E" prefix="yes">
  <MsgText pgmKey="error.calling.async.dispatch" varFormat="Java">
    error.calling.async.dispatch=SRVE8046E: Se ha producido un error al invocar una llamar para el dispatch de AsyncContext.
  </MsgText>
  <Explanation>
    Se ha producido un error al invocar una llamar para el dispatch de AsyncContext.
  </Explanation>
  <UserResponse>
    Consulte la causa raíz de la excepción.
  </UserResponse>
</Message>
<Message ID="SRVE0285E" severity="E" prefix="yes">
  <MsgText pgmKey="exception.caught.destroying.context" varFormat="Java">
    exception.caught.destroying.context=SRVE0285E: Se ha detectado una excepción al destruir el contexto: {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE8003E" severity="E" prefix="yes">
  <MsgText pgmKey="add.servlet.mapping.to.null.url.patterns" varFormat="Java">
    add.servlet.mapping.to.null.url.patterns=SRVE8003E: Se ha intentado añadir una correlación de servlets con el parámetro de patrón URL del argumento variable establecido en nulo.
  </MsgText>
  <Explanation>
    El parámetro de patrón URL del argumento variable pasado a ServletRegistration.addMapping no debe ser nulo ni debe estar vacío.
  </Explanation>
  <UserResponse>
    Modifique el código de aplicación de acuerdo con la explicación del error.
  </UserResponse>
</Message>
<Message ID="SRVE8095W" severity="W" prefix="yes">
  <MsgText pgmKey="async.servlet.work.rejected" varFormat="Java">
    async.servlet.work.rejected=SRVE8095W: Se ha llamado a workRejected para el ejecutable [{0}].
  </MsgText>
  <Explanation>
    Al planificar un elemento de trabajo con un gestor de trabajo de bean asíncrono, es posible que se llame a workRejected si se produce algún error al iniciar el elemento de trabajo.
  </Explanation>
  <UserResponse>
    Consulte la causa raíz.
  </UserResponse>
</Message>
<Message ID="SRVE0282E" severity="E" prefix="yes">
  <MsgText pgmKey="error.while.setting.WebAppAttributes" varFormat="Java">
    error.while.setting.WebAppAttributes=SRVE0282E: Se ha producido un error al establecer los atributos de la aplicación web: {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0252W" severity="W" prefix="yes">
  <MsgText pgmKey="transports.detected" varFormat="Java">
    transports.detected=SRVE0252W: Se han detectado transportes y cadenas. Se han modificado los transportes para utilizar un nuevo modelo. Utilice los programas de utilidad de migración para migrar los transporte al modelo nuevo. La configuración de agrupaciones de hebras en el contenedor web no se utilizará con estos transportes.
  </MsgText>
  <Explanation>
    Los transportes se han utilizado para que utilicen cadenas de canal. Consulte el Information Center para revisar la información sobre cadenas de canal y sobre cómo migrar los transportes existentes para que se basen en una cadena de canal. La configuración de agrupaciones de hebras en el contenedor web no se utilizará en estos transportes.
  </Explanation>
  <UserResponse>
    Utilice los programas de utilidad de migración para migrar el modelo de configuración desde los transportes a las nuevas cadenas de canal.
  </UserResponse>
</Message>
<Message ID="SRVE0220I" severity="I" prefix="yes">
  <MsgText pgmKey="Wrapped.Error" varFormat="Java">
    Wrapped.Error=SRVE0220I: Error de acomodación
  </MsgText>
  <Explanation>
    Error que ha causado el problema.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8052E" severity="E" prefix="yes">
  <MsgText pgmKey="classnotfoundexception.loading.servlet.class" varFormat="Java">
    classnotfoundexception.loading.servlet.class=SRVE8052E: Registrando ClassNotFoundException
  </MsgText>
  <Explanation>
    El servlet no ha podido encontrar la clase necesaria para servir a la solicitud. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Verifique que exista la clase dentro de la aplicación.
  </UserResponse>
</Message>
<Message ID="SRVE0014E" severity="E" prefix="yes">
  <MsgText pgmKey="Uncaught.service().exception.root.cause" varFormat="Java">
    Uncaught.service().exception.root.cause=SRVE0014E: No se ha podido detectar la causa raíz {0}: {1} de la excepción service()
  </MsgText>
  <Explanation>
    No se ha obtenido la excepción creada en el método de servicio del servlet. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Varía en función de la causa raíz.
  </UserResponse>
</Message>
<Message ID="SRVE8073E" severity="E" prefix="yes">
  <MsgText pgmKey="Null.webApp.configuration.object" varFormat="Java">
    Null.webApp.configuration.object=SRVE8073E: Objeto de configuración de aplicación web nulo.
  </MsgText>
  <Explanation>
    Se ha producido una excepción de configuración inesperada.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0185E" severity="E" prefix="yes">
  <MsgText pgmKey="error.occurred.processing.request" varFormat="Java">
    error.occurred.processing.request=SRVE0185E: Se ha producido un error al procesar la petición:
  </MsgText>
  <Explanation>
    Se ha producido una excepción al procesar la solicitud.
  </Explanation>
  <UserResponse>
    Revise el mensaje para determinar la causa raíz del problema.
  </UserResponse>
</Message>
<Message ID="SRVE0070E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.Initializing.for.Next.Request" varFormat="Java">
    Error.Initializing.for.Next.Request=SRVE0070E: Error al inicializar para siguiente solicitud
  </MsgText>
  <Explanation>
    Error interno de contenedor Web.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0199E" severity="E" prefix="yes">
  <MsgText pgmKey="OutputStream.already.obtained" varFormat="Java">
    OutputStream.already.obtained=SRVE0199E: Ya se ha obtenido OutputStream
  </MsgText>
  <Explanation>
    La aplicación ha solicitado un PrintWriter después de haber obtenido ya ServletResponse OutputStream. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Revise la aplicación para determinar si se ha obtenido tanto PrintWriter como OuputStream. Sólo modifique el servlet/JSP que falla para obtener el uno o el otro.
  </UserResponse>
</Message>
<Message ID="SRVE8036E" severity="E" prefix="yes">
  <MsgText pgmKey="InstantiationException.loading.servlet" varFormat="Java">
    InstantiationException.loading.servlet=SRVE8036E: Servlet [{0}]: {1} no se ha podido cargar debido a una InstantiationException.
  </MsgText>
  <Explanation>
    Se produce una InstantiationException al intentar crear una instancia nueva de un interfaz o clase abstracta.
  </Explanation>
  <UserResponse>
    Cambie la clase que se está instanciando por una clase concreta o elimine la definición como de servlet.
  </UserResponse>
</Message>
<Message ID="SRVE0053E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.Argument:.Invalid.Date.Format" varFormat="Java">
    Illegal.Argument:.Invalid.Date.Format=SRVE0053E: Excepción de argumento no permitido: Formato de fecha no válido
  </MsgText>
  <Explanation>
    El formato de fecha no es válido.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema. Si no existen mensajes relacionados o no ayudan a resolver el problema, póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0262E" severity="E" prefix="yes">
  <MsgText pgmKey="annotation.helper.wrong.type" varFormat="Java">
    annotation.helper.wrong.type=SRVE0262E: El ayudante de anotación {0} no es del tipo com.ibm.wsspi.webcontainer.AnnotationHelper.
  </MsgText>
  <Explanation>
    El ayudante de anotación especificado no es del tipo esperado. Es un error del sistema.
  </Explanation>
  <UserResponse>
    Compruebe la clase de implementación del ayudante de anotación y arréglela.
  </UserResponse>
</Message>
<Message ID="SRVE8103E" severity="E" prefix="yes">
  <MsgText pgmKey="Exception.creating.channel" varFormat="Java">
    Exception.creating.channel=SRVE8103E: Se ha producido un error de servidor interno al crear el objeto de canal.
  </MsgText>
  <Explanation>
    Se ha producido un error de servidor interno al crear el objeto de canal.
  </Explanation>
  <UserResponse>
    Recopile registros de servidor y póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8057E" severity="E" prefix="yes">
  <MsgText pgmKey="error.initializing.output.stream" varFormat="Java">
    error.initializing.output.stream=SRVE8057E: Se ha producido una excepción inesperada al inicializar la secuencia de salida.
  </MsgText>
  <Explanation>
    Se ha producido una excepción inesperada al inicializar la secuencia de salida.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0121E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.Argument:.Trying.to.write.&lt;.0.bytes" varFormat="Java">
    Illegal.Argument:.Trying.to.write.&lt;.0.bytes=SRVE0121E: Excepción de argumento no permitido: Se están intentando escribir &amp;lt; 0 bytes
  </MsgText>
  <Explanation>
    Se ha intentando escribir un almacenamiento intermedio de un tamaño inferior a 0. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Sólo pueden escribirse almacenamientos intermedios de un tamaño 0 o superior.
  </UserResponse>
</Message>
<Message ID="SRVE8094W" severity="W" prefix="yes">
  <MsgText pgmKey="Cannot.set.header.Response.already.committed" varFormat="Java">
    Cannot.set.header.Response.already.committed=SRVE8094W: AVISO: No se puede establecer la cabecera. Ya se confirmado la respuesta.
  </MsgText>
  <Explanation>
    Una vez se confirma la respuesta, ya no se pueden definir más cabeceras de respuesta.
  </Explanation>
  <UserResponse>
    Verifique si esta llamada es necesaria. Si la cabecera realmente debe enviarse, debería establecerse antes de confirmar la respuesta.
  </UserResponse>
</Message>
<Message ID="SRVE0312E" severity="E" prefix="yes">
  <MsgText pgmKey="error.adding.channel" varFormat="Java">
    error.adding.channel=SRVE0312E: Se ha producido un error al añadir un canal: {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0317W" severity="W" prefix="yes">
  <MsgText pgmKey="need.to.add.a.new.virtual.host.alias" varFormat="Java">
    need.to.add.a.new.virtual.host.alias=SRVE0317W: Es posible que deba añadir un nuevo alias de host virtual de *:&lt;su puerto&gt; al mismo host virtual bajo el que está [{0}].
  </MsgText>
  <Explanation>
    La vía de acceso de solicitud coincide con una raíz de contexto en un host virtual no coincidente.
  </Explanation>
  <UserResponse>
    Siga las instrucciones en el Information Center para añadir un nuevo alias de host virtual.
  </UserResponse>
</Message>
<Message ID="SRVE8000W" severity="W" prefix="yes">
  <MsgText pgmKey="ClassNotFound.check.web.xml" varFormat="Java">
    ClassNotFound.check.web.xml=SRVE8000W: Se ha saltado la clase que no ha podido inicializarse para explorar la anotación.
  </MsgText>
  <Explanation>
    Una clase del módulo web no se ha podido inicializar, por lo que no puede explorarse para anotaciones.
  </Explanation>
  <UserResponse>
    Verifique que exista la clase y que tenga todas sus dependencias en la vía de acceso de clases.
  </UserResponse>
</Message>
<Message ID="SRVE8020E" severity="E" prefix="yes">
  <MsgText pgmKey="multipart.no.multipart.config" varFormat="Java">
    multipart.no.multipart.config=SRVE8020E: El servlet no acepta solicitudes en varias partes
  </MsgText>
  <Explanation>
    El servlet no acepta solicitudes en varias partes.
  </Explanation>
  <UserResponse>
    Añada una anotación @MultipartConfig a este servlet o añada un elemento configurado en varias partes a este servlet en el web.xml.
  </UserResponse>
</Message>
<Message ID="SRVE8090E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.occurred.while.preparing.the.servlet.for.initialization" varFormat="Java">
    Error.occurred.while.preparing.the.servlet.for.initialization=SRVE8090E: Se ha producido un error al preparar el derivador de servlet para la inicialización.
  </MsgText>
  <Explanation>
    Se ha producido un error al preparar el derivador de servlet para la inicialización.
  </Explanation>
  <UserResponse>
    Consulte la causa raíz.
  </UserResponse>
</Message>
<Message ID="SRVE0322E" severity="E" prefix="yes">
  <MsgText pgmKey="Listener.found.but.injection.failure" varFormat="Java">
    Listener.found.but.injection.failure=SRVE0322E: Se ha encontrado la clase de escucha [{0}], pero se ha producido una anomalía en la inyección de recursos.
  </MsgText>
  <Explanation>
    La escucha no ha podido inicializarse debido a que se ha producido un error cuando se inyectaba un recurso.
  </Explanation>
  <UserResponse>
    Verifique que existan los recursos inyectados y que sean accesibles para la escucha. Para volver a iniciar el escucha, reinicie la aplicación de escucha.
  </UserResponse>
</Message>
<Message ID="SRVE8026E" severity="E" prefix="yes">
  <MsgText pgmKey="thread.interrupted.scheduling.async.runnable.on.thread.pool" varFormat="Java">
    thread.interrupted.scheduling.async.runnable.on.thread.pool=SRVE8026E: Se ha producido una excepción al planificar el trabajo relacionado con el proceso del servlet asíncrono.
  </MsgText>
  <Explanation>
    Esta excepción no debería suceder nunca pues las agrupaciones de hebras pondrán el trabajo en cola.
  </Explanation>
  <UserResponse>
    Si se produce esta excepción, póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8108W" severity="W" prefix="yes">
  <MsgText pgmKey="trying.to.set.number.of.async.timer.threads.to.less.than.one" varFormat="Java">
    trying.to.set.number.of.async.timer.threads.to.less.than.one=SRVE8108W: Intentando establecer el número de hebras de temporización asíncrona a menos de una.
  </MsgText>
  <Explanation>
    Debe haber cómo mínimo una hebra de temporización asíncrona.
  </Explanation>
  <UserResponse>
    Compruebe su configuración de servidor para ver si el valor está establecido.
  </UserResponse>
</Message>
<Message ID="SRVE0011E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.Argument.Trying.to.write.chars" varFormat="Java">
    Illegal.Argument.Trying.to.write.chars=SRVE0011E: Excepción de argumento no permitido: Se están intentando escribir &lt; 0 caracteres
  </MsgText>
  <Explanation>
    Se ha intentando escribir un almacenamiento intermedio de un tamaño inferior a 0. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Sólo pueden escribirse almacenamientos intermedios de un tamaño 0 o superior.
  </UserResponse>
</Message>
<Message ID="SRVE0255E" severity="E" prefix="yes">
  <MsgText pgmKey="Web.Group.VHost.Not.Found" varFormat="Java">
    Web.Group.VHost.Not.Found=SRVE0255E: No se ha definido un host WebGroup/Virtual para manejar {0}.
  </MsgText>
  <Explanation>
    No se ha podido encontrar un grupo web (módulo web) ni un host virtual para manejar la solicitud. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Asegúrese de que el grupo web y el host virtual se definan y se desplieguen.
  </UserResponse>
</Message>
<Message ID="SRVE0279E" severity="E" prefix="yes">
  <MsgText pgmKey="error.processing.global.listeners.for.webapp" varFormat="Java">
    error.processing.global.listeners.for.webapp=SRVE0279E: Se ha producido un error al procesar escuchas globales para la aplicación {0}: {1}
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0238E" severity="E" prefix="yes">
  <MsgText pgmKey="resource.path.has.to.start.with.slash" varFormat="Java">
    resource.path.has.to.start.with.slash=SRVE0238E: Las vías de acceso de recurso deben tener una barra inclinada principal
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Proporcione una barra inclinada principal antes de la vía de acceso y vuelva a intentarlo.
  </UserResponse>
</Message>
<Message ID="SRVE0140E" severity="E" prefix="yes">
  <MsgText pgmKey="instantiate.collaborator.exception" varFormat="Java">
    instantiate.collaborator.exception=SRVE0140E: No se ha podido crear una instancia del colaborador de seguridad {0}
  </MsgText>
  <Explanation>
    Error interno del servidor.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8048E" severity="E" prefix="yes">
  <MsgText pgmKey="mapping.clash.occurred" varFormat="Java">
    mapping.clash.occurred=SRVE8048E: Se ha producido un conflicto de correlación al añadir el patrón de URL [{0}]
  </MsgText>
  <Explanation>
    Se ha especificado el mismo patrón dos veces para que los patrones de URL denegaran el servicio de archivos.
  </Explanation>
  <UserResponse>
    Compruebe los atributos de servicio de archivos en el archivo ibm-web-ext.xmi oibm-web-ext.xml.
  </UserResponse>
</Message>
<Message ID="SRVE0272W" severity="W" prefix="yes">
  <MsgText pgmKey="jsp.processor.not.defined.skipping" varFormat="Java">
    jsp.processor.not.defined.skipping=SRVE0272W: No se ha definido el procesador JSP. Se saltará : {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8017E" severity="E" prefix="yes">
  <MsgText pgmKey="error.validate.webServlet.urlPatterns.and.value" varFormat="Java">
    error.validate.webServlet.urlPatterns.and.value=SRVE8017E: La clase {0} no puede especificar tanto urlPatterns como el valor en la anotación
  </MsgText>
  <Explanation>
    La clase no puede especificar tanto urlPatterns como el valor en la anotación
  </Explanation>
  <UserResponse>
    Modifique la clase para que sólo incluya uno de los atributos de valor o urlPatterns
  </UserResponse>
</Message>
<Message ID="SRVE0225I" severity="I" prefix="yes">
  <MsgText pgmKey="Root.Error" varFormat="Java">
    Root.Error=SRVE0225I: Error de root
  </MsgText>
  <Explanation>
    Ninguna.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0160E" severity="E" prefix="yes">
  <MsgText pgmKey="usertransaction.rollback.exception" varFormat="Java">
    usertransaction.rollback.exception=SRVE0160E: Excepción al retrotraer UserTransaction: {0}
  </MsgText>
  <Explanation>
    Problema encontrado al retrotraer la transacción de usuario especificada. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema. Si no existen mensajes relacionados o no ayudan a resolver el problema, póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0108E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.Argument:.Missing.flag.value" varFormat="Java">
    Illegal.Argument:.Missing.flag.value=SRVE0108E: Excepción de argumento no permitido: Falta valor de distintivo
  </MsgText>
  <Explanation>
    Falta valor de distintivo.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema. Si no existen mensajes relacionados o no ayudan a resolver el problema, póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8056E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.while.closing.response.output" varFormat="Java">
    Error.while.closing.response.output=SRVE8056E: Se ha producido una excepción inesperada al cerrar la secuencia de salida.
  </MsgText>
  <Explanation>
    Se ha producido una excepción inesperada al cerrar la secuencia de salida.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0313E" severity="E" prefix="yes">
  <MsgText pgmKey="transport.migration.error" varFormat="Java">
    transport.migration.error=SRVE0313E: Se ha producido un error al migrar el transporte del puerto {0} a la correspondiente cadena: {1}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0239I" severity="I" prefix="yes">
  <MsgText pgmKey="ExtensionFactory.[{0}].registered.successfully" varFormat="Java">
    ExtensionFactory.[{0}].registered.successfully=SRVE0239I: Extension Factory [{0}] se ha registrado satisfactoriamente.
  </MsgText>
  <Explanation>
    Ninguna.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0201E" severity="E" prefix="yes">
  <MsgText pgmKey="Servlet.not.a.servlet.class" varFormat="Java">
    Servlet.not.a.servlet.class=SRVE0201E: Servlet [{0}]: no es una clase de servlet
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Modifique la clase que falla para implementar javax.servlet.Servlet o ampliar javax.servlet.GenericServlet o javax.servlet.http.HttpServlet
  </UserResponse>
</Message>
<Message ID="SRVE0291E" severity="E" prefix="yes">
  <MsgText pgmKey="unable.to.use.default.temp.directory" varFormat="Java">
    unable.to.use.default.temp.directory=SRVE0291E: No se puede utilizar el directorio temporal predeterminado: {0} leer:{1} escribir:{2}
  </MsgText>
  <Explanation>
    Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Compruebe el estado del directorio, incluyendo la existencia, los privilegios de acceso y el espacio disponible.
  </UserResponse>
</Message>
<Message ID="SRVE0214E" severity="E" prefix="yes">
  <MsgText pgmKey="invalid.count" varFormat="Java">
    invalid.count=SRVE0214E: recuento no válido
  </MsgText>
  <Explanation>
    Excepción interna.
  </Explanation>
  <UserResponse>
    Si el problema persiste, puede haber información adicional disponible si busca el ID del mensaje en los sitios Web siguientes: página de soporte de IBM WebSphere Application Server: http://www.ibm.com/software/webservers/appserv/was/support/   página de soporte de IBM WebSphere Application Server for z/OS: http://www.ibm.com/software/webservers/appserv/zos_os390/support/
  </UserResponse>
</Message>
<Message ID="SRVE0271E" severity="E" prefix="yes">
  <MsgText pgmKey="uncaught.init.exception.thrown.by.servlet" varFormat="Java">
    uncaught.init.exception.thrown.by.servlet=SRVE0271E: El servlet [{0}] no ha detectado la excepción init() en la aplicación [{1}]: {2}
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0096E" severity="E" prefix="yes">
  <MsgText pgmKey="Session.Feature.Set.Not.Started" varFormat="Java">
    Session.Feature.Set.Not.Started=SRVE0096E: No se puede acceder a una sesión.  El conjunto de características del gestor de sesiones no se ha iniciado.
  </MsgText>
  <Explanation>
    No se puede acceder a una sesión.  El conjunto de características del gestor de sesiones no se ha iniciado.
  </Explanation>
  <UserResponse>
    Asegúrese de que la configuración de servidor dependa del conjunto de características de la sesión.
  </UserResponse>
</Message>
<Message ID="SRVE8008E" severity="E" prefix="yes">
  <MsgText pgmKey="add.filter.mapping.to.empty.servlet.names" varFormat="Java">
    add.filter.mapping.to.empty.servlet.names=SRVE8008E: Se ha intentado añadir una correlación de filtros con un parámetro de nombre de servlet de argumento variable vacío.
  </MsgText>
  <Explanation>
    El parámetro de nombre de servlet de argumento variable pasado a FilterRegistration.addMappingForServletNames no debe ser nulo ni debe estar vacío.
  </Explanation>
  <UserResponse>
    Modifique el código de aplicación de acuerdo con la explicación del error.
  </UserResponse>
</Message>
<Message ID="SRVE8087E" severity="E" prefix="yes">
  <MsgText pgmKey="error.creating.stats.instance" varFormat="Java">
    error.creating.stats.instance=SRVE8087E: Se ha producido un error al crear la instancia de estadísticas de Performance Monitoring Infrastructure (PMI).
  </MsgText>
  <Explanation>
    El método createStatsInstance() ha creado una excepción cuando se le ha invocado para crear la instancia de estadísticas. No se han recopilado estadísticas.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema.
  </UserResponse>
</Message>
<Message ID="SRVE0229E" severity="E" prefix="yes">
  <MsgText pgmKey="class.transfered.using.binary.mode" varFormat="Java">
    class.transfered.using.binary.mode=SRVE0229E: Compruebe que la clase se haya transferido al sistema de archivos utilizando un nodo de transferencia binario.
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Compruebe que la clase se haya transferido al sistema de archivos utilizando un nodo de transferencia binario.
  </UserResponse>
</Message>
<Message ID="SRVE0095I" severity="I" prefix="yes">
  <MsgText pgmKey="Servlet.has.become.temporarily.unavailable.for.service:.{0}" varFormat="Java">
    Servlet.has.become.temporarily.unavailable.for.service:.{0}=SRVE0095I: El servlet ha pasado a no estar disponible temporalmente para el servicio: {0}
  </MsgText>
  <Explanation>
    El servlet especificado no está disponible temporalmente.
  </Explanation>
  <UserResponse>
    Espere y vuelta a intentar la operación.  Es posible que el administrador del sistema haya eliminado el servlet del servicio.
  </UserResponse>
</Message>
<Message ID="SRVE8100W" severity="W" prefix="yes">
  <MsgText pgmKey="virtual.host.alias.clash" varFormat="Java">
    virtual.host.alias.clash=SRVE8100W: El alias VirtualHost ya existe para este VirtualHost --&gt;[{0}].
  </MsgText>
  <Explanation>
    Se ha producido un error al añadir un alias de host virtual porque ya existía uno con el mismo patrón.
  </Explanation>
  <UserResponse>
    Edite su alias de host virtual y elimine todos los duplicados.
  </UserResponse>
</Message>
<Message ID="SRVE0157E" severity="E" prefix="yes">
  <MsgText pgmKey="setbuffer.size.called.after.write" varFormat="Java">
    setbuffer.size.called.after.write=SRVE0157E: setBufferSize() llamada tras la primera grabación en la corriente de salida/grabador
  </MsgText>
  <Explanation>
    Se ha intentado establecer el tamaño del almacenamiento intermedio de respuesta después de que se haya comprometido la respuesta. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Elimine la llamada a setBufferSize() o muévalo a una posición anterior al compromiso de la respuesta.
  </UserResponse>
</Message>
<Message ID="SRVE0267E" severity="E" prefix="yes">
  <MsgText pgmKey="error.initializing.filters" varFormat="Java">
    error.initializing.filters=SRVE0267E: Se ha producido un error al inicializar los filtros: {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE8065E" severity="E" prefix="yes">
  <MsgText pgmKey="webcontianer.configuration.error" varFormat="Java">
    webcontianer.configuration.error=SRVE8065E: Se ha producido un error desconocido relacionado con la configuración del contenedor web.
  </MsgText>
  <Explanation>
    Se ha producido un error desconocido relacionado con la configuración del contenedor web.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8059E" severity="E" prefix="yes">
  <MsgText pgmKey="unable.to.get.sessionContext" varFormat="Java">
    unable.to.get.sessionContext=SRVE8059E: Se ha producido una excepción inesperada al intentar recuperar el contexto de sesión
  </MsgText>
  <Explanation>
    Se ha producido una excepción inesperada al intentar recuperar el contexto de sesión
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0236E" severity="E" prefix="yes">
  <MsgText pgmKey="error.on.collaborator.starting.call" varFormat="Java">
    error.on.collaborator.starting.call=SRVE0236E: Se ha producido un error al invocar el colaborador de inicialización en la llamada starting()
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Compruebe los registros de captura de datos en primer error para obtener más detalles sobre el error.
  </UserResponse>
</Message>
<Message ID="SRVE0041E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.Argument:.Bootstrap.File.Not.Found" varFormat="Java">
    Illegal.Argument:.Bootstrap.File.Not.Found=SRVE0041E: Excepción de argumento no permitido: No se ha encontrado el archivo de rutina de carga
  </MsgText>
  <Explanation>
    Este es un error interno del sistema y está causado probablemente por una anomalía anterior.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema. Si no existen mensajes relacionados o no ayudan a resolver el problema, póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8097W" severity="W" prefix="yes">
  <MsgText pgmKey="null.url.pattern.and.servlet.name.for.filter.mapping" varFormat="Java">
    null.url.pattern.and.servlet.name.for.filter.mapping=SRVE8097W: No hay ningún patrón de URL no nombre de servlet para la correlación de filtros del filtro denominado [{0}].
  </MsgText>
  <Explanation>
    Una correlación de filtros está incompleta sin un patrón de URL o nombre de servlet con el que deba correlacionarse.
  </Explanation>
  <UserResponse>
    Añada un patrón de URL o un nombre de servlet para filtrar la correlación.
  </UserResponse>
</Message>
<Message ID="SRVE8111E" severity="E" prefix="yes">
  <MsgText pgmKey="programmatic.session.cookie.restricted.match" varFormat="Java">
    programmatic.session.cookie.restricted.match=SRVE8111E: La aplicación {0} está intentando modificar una cookie que coincide con un patrón en la lista de cookies de sesión programática restringida [dominio={1}, nombre={2}, vía de acceso={3}].
  </MsgText>
  <Explanation>
    El administrador ha inhabilitado el cambio de configuración de cookies para la cookie que coincide con el dominio, nombre y vía de acceso.
  </Explanation>
  <UserResponse>
    Modifique la aplicación para que utilice un nombre de cookie, dominio y vía de acceso diferentes o modifique los valores de cookies de sesión programática restringida
  </UserResponse>
</Message>
<Message ID="SRVE0256E" severity="E" prefix="yes">
  <MsgText pgmKey="webcontainer.not.initialized" varFormat="Java">
    webcontainer.not.initialized=SRVE0256E: El contenedor web no se ha inicializado.
  </MsgText>
  <Explanation>
    El contenedor web no se inicializa hasta que no se haya inicializado una aplicación. Es un error del sistema.
  </Explanation>
  <UserResponse>
    Instale una aplicación para inicializar el contenedor web.
  </UserResponse>
</Message>
<Message ID="SRVE0233E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.Report" varFormat="Java">
    Error.Report=SRVE0233E: Informe de errores
  </MsgText>
  <Explanation>
    ninguna
  </Explanation>
  <UserResponse>
    ninguna
  </UserResponse>
</Message>
<Message ID="SRVE0283E" severity="E" prefix="yes">
  <MsgText pgmKey="exception.while.initializing.context" varFormat="Java">
    exception.while.initializing.context=SRVE0283E: Se ha detectado una excepción al inicializar el contexto: {0}
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0237E" severity="E" prefix="yes">
  <MsgText pgmKey="error.on.collaborator.started.call" varFormat="Java">
    error.on.collaborator.started.call=SRVE0237E: Se ha producido un error al invocar el colaborador de inicialización en la llamada started()
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Compruebe los registros de captura de datos en primer error para obtener más detalles sobre el error
  </UserResponse>
</Message>
<Message ID="SRVE8035E" severity="E" prefix="yes">
  <MsgText pgmKey="IllegalAccessException.loading.servlet" varFormat="Java">
    IllegalAccessException.loading.servlet=SRVE8035E: Servlet [{0}]: {1} no se ha podido cargar debido a una IllegalAccessException.
  </MsgText>
  <Explanation>
    Se ha producido una IllegalAccessException llamando a newInstance en el servlet.
  </Explanation>
  <UserResponse>
    Verifique que no haya ningún constructor de argumentos en el servlet.
  </UserResponse>
</Message>
<Message ID="SRVE8063E" severity="E" prefix="yes">
  <MsgText pgmKey="Could.not.create.LoginProcessor" varFormat="Java">
    Could.not.create.LoginProcessor=SRVE8063E: Se ha producido una excepción inesperada al inicializar el procesador de inicio de sesión.
  </MsgText>
  <Explanation>
    Se ha producido una excepción inesperada al inicializar el procesador de inicio de sesión.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0180I" severity="I" prefix="yes">
  <MsgText pgmKey="[{0}][{1}][Servlet.LOG]:.{2}" varFormat="Java">
    [{0}][{1}][Servlet.LOG]:.{2}=SRVE0180I: [{0}] [{1}] [Servlet.LOG]: {2}
  </MsgText>
  <Explanation>
    Mensaje informativo anotado cronológicamente a través de ServletContext.log desde una aplicación.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0216E" severity="E" prefix="yes">
  <MsgText pgmKey="post.body.contains.less.bytes.than.specified" varFormat="Java">
    post.body.contains.less.bytes.than.specified=SRVE0216E: la petición enviada contiene menos bytes que los especificados en la longitud del contenido
  </MsgText>
  <Explanation>
    Es posible que se haya corrompido la solicitud entrante. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8067E" severity="E" prefix="yes">
  <MsgText pgmKey="Failed.to.handle.extension.point" varFormat="Java">
    Failed.to.handle.extension.point=SRVE8067E: Se ha producido una excepción inesperada al manejar un punto de ampliación.
  </MsgText>
  <Explanation>
    Se ha producido una excepción inesperada al manejar un punto de ampliación.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8058E" severity="E" prefix="yes">
  <MsgText pgmKey="Did.not.have.access.to.delete.Directory" varFormat="Java">
    Did.not.have.access.to.delete.Directory=SRVE8058E: Sin acceso para suprimir el directorio temporal
  </MsgText>
  <Explanation>
    La política de seguridad Java 2 ha impedido la supresión del directorio temporal
  </Explanation>
  <UserResponse>
    Compruebe la política de seguridad de Java 2 para verificar si debe permitirse este acceso.
  </UserResponse>
</Message>
<Message ID="SRVE8015E" severity="E" prefix="yes">
  <MsgText pgmKey="exception.occurred.while.adding.listener" varFormat="Java">
    exception.occurred.while.adding.listener=SRVE8015E: Se ha producido una excepción al intentar añadir un escucha del tipo {0}
  </MsgText>
  <Explanation>
    Se ha producido una excepción al intentar añadir un escucha
  </Explanation>
  <UserResponse>
    Verifique que el escucha sea de un tipo válido y que esté disponible en la aplicación
  </UserResponse>
</Message>
<Message ID="SRVE0163I" severity="I" prefix="yes">
  <MsgText pgmKey="web.container.jsp.spec.level" varFormat="Java">
    web.container.jsp.spec.level=SRVE0163I: Nivel de especificación de JavaServer Pages soportado: 2.2
  </MsgText>
  <Explanation>
    El nivel de especificación de JSP soportado por el servidor de aplicaciones.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8077E" severity="E" prefix="yes">
  <MsgText pgmKey="Null.webApp.config.object" varFormat="Java">
    Null.webApp.config.object=SRVE8077E: Objeto de configuración de aplicación web nulo.
  </MsgText>
  <Explanation>
    Se ha producido una excepción de configuración inesperada.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0066I" severity="I" prefix="yes">
  <MsgText pgmKey="Waiting.servlet.to.finish.servicing.requests:.{0}" varFormat="Java">
    Waiting.servlet.to.finish.servicing.requests:.{0}=SRVE0066I: Se está esperando a que el servlet termine de atender las solicitudes: {0}
  </MsgText>
  <Explanation>
    El servlet aún está procesando la solicitud. Espere a que finalice la solicitud antes de llamar al método destroy.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8109W" severity="W" prefix="yes">
  <MsgText pgmKey="uncaught.filter.exception" varFormat="Java">
    uncaught.filter.exception=SRVE8109W: No se ha obtenido la excepción generada por el filtro {0}: {1}
  </MsgText>
  <Explanation>
    Se ha generado una excepción desde un filtro.
  </Explanation>
  <UserResponse>
    Consulte la causa raíz.
  </UserResponse>
</Message>
<Message ID="SRVE0188E" severity="E" prefix="yes">
  <MsgText pgmKey="Class.{0}.does.not.implement.servlet" varFormat="Java">
    Class.{0}.does.not.implement.servlet=SRVE0188E: La clase {0} no implementa el servlet
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    La clase mencionada necesita implementar javax.servlet.Servlet o ampliar javax.servlet.GenericServlet o javax.servlet.http.HttpServlet
  </UserResponse>
</Message>
<Message ID="SRVE0231E" severity="E" prefix="yes">
  <MsgText pgmKey="class.not.renamed.after.compiled" varFormat="Java">
    class.not.renamed.after.compiled=SRVE0231E: Compruebe que no se haya cambiado el nombre del archivo de clase una vez compilado.
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Compruebe que no se haya cambiado el nombre del archivo de clase una vez compilado.
  </UserResponse>
</Message>
<Message ID="SRVE0319E" severity="E" prefix="yes">
  <MsgText pgmKey="Servlet.found.but.injection.failure" varFormat="Java">
    Servlet.found.but.injection.failure=SRVE0319E: Para el servlet [{0}], se ha encontrado la clase de servlet {1} pero se ha producido una anomalía en la inyección  de recursos.
  </MsgText>
  <Explanation>
    El servlet no ha podido inicializarse debido a que se ha producido un error cuando se inyectaba un recurso.
  </Explanation>
  <UserResponse>
    Verifique que existan los recursos inyectados y que sean accesibles para el servlet.
  </UserResponse>
</Message>
<Message ID="SRVE0241I" severity="I" prefix="yes">
  <MsgText pgmKey="Using.[{0}].as.server.root" varFormat="Java">
    Using.[{0}].as.server.root=SRVE0241I: Se está utilizando [{0}] como servidor raíz en getTempDirectory().
  </MsgText>
  <Explanation>
    Ninguna.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8086E" severity="E" prefix="yes">
  <MsgText pgmKey="invalid.peer.certificate" varFormat="Java">
    invalid.peer.certificate=SRVE8086E: Certificado igual no válido.
  </MsgText>
  <Explanation>
    Se ha producido una excepción al recuperar el certificado igual.
  </Explanation>
  <UserResponse>
    Consulte la causa raíz.
  </UserResponse>
</Message>
<Message ID="SRVE0055I" severity="I" prefix="yes">
  <MsgText pgmKey="Servlet.wait.for.destroy.timeout.has.expired,.destroy.will.be.forced:.{0}" varFormat="Java">
    Servlet.wait.for.destroy.timeout.has.expired,.destroy.will.be.forced:.{0}=SRVE0055I: El tiempo de espera del servlet para la destrucción ha finalizado; se forzará la destrucción: {0}
  </MsgText>
  <Explanation>
    Se va a destruir el servlet después de que se haya excedido el tiempo de espera.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0318W" severity="W" prefix="yes">
  <MsgText pgmKey="invalid.query.string" varFormat="Java">
    invalid.query.string=SRVE0318W: Carácter no válido en la serie de consulta.
  </MsgText>
  <Explanation>
    Hay un carácter no es válido en la serie de consulta y se ha ignorado.
  </Explanation>
  <UserResponse>
    No se requiere ninguna acción por parte del usuario.
  </UserResponse>
</Message>
<Message ID="SRVE8014E" severity="E" prefix="yes">
  <MsgText pgmKey="exception.occurred.while.creating.listener.instance" varFormat="Java">
    exception.occurred.while.creating.listener.instance=SRVE8014E: Se ha producido una excepción al crear la instancia de escucha
  </MsgText>
  <Explanation>
    Se ha producido una excepción al crear la instancia de escucha
  </Explanation>
  <UserResponse>
    Se ha producido una excepción al crear la instancia de escucha
  </UserResponse>
</Message>
<Message ID="SRVE8047E" severity="E" prefix="yes">
  <MsgText pgmKey="NumberFormatException.for.file.size.at.which.you.switch.to.optimized.file.serving" varFormat="Java">
    NumberFormatException.for.file.size.at.which.you.switch.to.optimized.file.serving=SRVE8047E: NumberFormatException para el tamaño de archivo en el que se conmuta al servicio de archivos optimizado.
  </MsgText>
  <Explanation>
    El atributo de servicio de archivos com.ibm.ws.webcontainer.optimizefileservingsize no se ha podido analizar de forma correcta.
  </Explanation>
  <UserResponse>
    Cambie el valor del atributo para que sea un valor entero válido.
  </UserResponse>
</Message>
<Message ID="SRVE0187E" severity="E" prefix="yes">
  <MsgText pgmKey="Check.your.classpath.ensure.all.classes.present" varFormat="Java">
    Check.your.classpath.ensure.all.classes.present=SRVE0187E: Compruebe la vía de acceso de clases para asegurarse de que se encuentren todas las clases que precise el servlet.
  </MsgText>
  <Explanation>
    No se ha encontrado una clase necesaria. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Asegúrese de que la vía de acceso de clases refleje todas las clases a las que vaya a acceder
  </UserResponse>
</Message>
<Message ID="SRVE0015E" severity="E" prefix="yes">
  <MsgText pgmKey="Failed.to.initialize.webapp.{0}" varFormat="Java">
    Failed.to.initialize.webapp.{0}=SRVE0015E: Error al inicializar la aplicación web {0}
  </MsgText>
  <Explanation>
    No se ha podido inicializar la aplicación Web especificada en el mensaje. Consulte el registro cronológico de errores para obtener más detalles.
  </Explanation>
  <UserResponse>
    Consulte el registro cronológico de errores para determinar la causa raíz de este problema.
  </UserResponse>
</Message>
<Message ID="SRVE0286E" severity="E" prefix="yes">
  <MsgText pgmKey="exception.caught.in.notifyServletContextDestroyed" varFormat="Java">
    exception.caught.in.notifyServletContextDestroyed=SRVE0286E: Se ha detectado una excepción en notifyServletContextDestroyed: {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0263E" severity="E" prefix="yes">
  <MsgText pgmKey="annotation.helper.cannot.create" varFormat="Java">
    annotation.helper.cannot.create=SRVE0263E: El ayudante de anotación {0} no se puede instanciar.
  </MsgText>
  <Explanation>
    El ayudante de anotación especificado no se ha podido instanciar. Es un error del sistema.
  </Explanation>
  <UserResponse>
    Compruebe los registros de excepción para buscar la razón por la que el ayudante de de anotación no se ha podido instanciar.
  </UserResponse>
</Message>
<Message ID="SRVE8104E" severity="E" prefix="yes">
  <MsgText pgmKey="Servlet.Engine.Exception" varFormat="Java">
    Servlet.Engine.Exception=SRVE8104E: Error interno del servidor. Mensaje de excepción: [{0}]
  </MsgText>
  <Explanation>
    Se ha producido una excepción de motor de servlet inesperada. Es un error del sistema.
  </Explanation>
  <UserResponse>
    Compruebe los registros System.out y System.err para obtener más información sobre la excepción generada.
  </UserResponse>
</Message>
<Message ID="SRVE0278E" severity="E" prefix="yes">
  <MsgText pgmKey="error.adding.servlet.mapping.file.handler" varFormat="Java">
    error.adding.servlet.mapping.file.handler=SRVE0278E: Se ha producido un error al añadir la correlación de servlets --&gt; {0}.
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Establezca fileServingEnabled=false en el archivo ibm-web-ext.xmi, que se encuentra en la carpeta WEB-INF.
  </UserResponse>
</Message>
<Message ID="SRVE8074E" severity="E" prefix="yes">
  <MsgText pgmKey="Null.webApp.object" varFormat="Java">
    Null.webApp.object=SRVE8074E: Objeto de aplicación web nulo.
  </MsgText>
  <Explanation>
    Se ha producido una excepción de configuración inesperada.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0274W" severity="W" prefix="yes">
  <MsgText pgmKey="error.while.adding.servlet.mapping.for.path" varFormat="Java">
    error.while.adding.servlet.mapping.for.path=SRVE0274W: Se ha producido un error al añadir la correlación de servlets para path--&gt;{0}, wrapper--&gt;{1}, application--&gt;{2}.
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Compruebe si se ha producido un conflicto de correlaciones en webconfig.xml
  </UserResponse>
</Message>
<Message ID="SRVE0277W" severity="W" prefix="yes">
  <MsgText pgmKey="mapping.for.directorybrowsingservlet.already.exists" varFormat="Java">
    mapping.for.directorybrowsingservlet.already.exists=SRVE0277W: Ya existe una correlación para DirectoryBrowsingServlet
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Elimine la correlación duplicada.
  </UserResponse>
</Message>
<Message ID="SRVE8037E" severity="E" prefix="yes">
  <MsgText pgmKey="exception.handing.off.reference.data" varFormat="Java">
    exception.handing.off.reference.data=SRVE8037E: Se ha producido una excepción al procesar datos de referencia.
  </MsgText>
  <Explanation>
    Se ha producido una excepción al procesar los datos de referencia. La aplicación intentará iniciarse igualmente.
  </Explanation>
  <UserResponse>
    Investigue la causa raíz de la excepción y determine si la aplicación aún funciona de forma correcta a pesar de estas anomalías.
  </UserResponse>
</Message>
<Message ID="SRVE0295E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.reported.{0}" varFormat="Java">
    Error.reported.{0}=SRVE0295E: Error notificado: {0}
  </MsgText>
  <Explanation>
    Se ha llamado a HttpServletResponse.sendError con un código de estado pero sin explicación.
  </Explanation>
  <UserResponse>
    Revise los registros para determinar el código de estado del error.
  </UserResponse>
</Message>
<Message ID="SRVE0056E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.Argument:.Unsupported.flag" varFormat="Java">
    Illegal.Argument:.Unsupported.flag=SRVE0056E: Excepción de argumento no permitido: Distintivo no soportado
  </MsgText>
  <Explanation>
    Este es un error interno del sistema y está causado probablemente por una anomalía anterior.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema. Si no existen mensajes relacionados o no ayudan a resolver el problema, póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0164E" severity="E" prefix="yes">
  <MsgText pgmKey="context.root.already.in.use" varFormat="Java">
    context.root.already.in.use=SRVE0164E: La aplicación web {0} utiliza la raíz de contexto {1} que ya está siendo utilizada por la aplicación web {2}. La aplicación Web {3} no se cargará.
  </MsgText>
  <Explanation>
    Hay dos aplicaciones Web en el mismo host virtual que comparten el mismo directorio raíz de contexto. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Cambie el directorio raíz de contexto de una de las aplicaciones Web o mueva una de ellas a un host virtual diferente.
  </UserResponse>
</Message>
<Message ID="SRVE8088E" severity="E" prefix="yes">
  <MsgText pgmKey="failed.to.remove.pmi.stats" varFormat="Java">
    failed.to.remove.pmi.stats=SRVE8088E: Se ha producido un error al eliminar la instancia de estadísticas de Performance Monitoring Infrastructure (PMI).
  </MsgText>
  <Explanation>
    Se ha producido un error al eliminar la instancia de estadísticas de Performance Monitoring Infrastructure (PMI).
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8019E" severity="E" prefix="yes">
  <MsgText pgmKey="error.validate.webServlet.name.empty" varFormat="Java">
    error.validate.webServlet.name.empty=SRVE8019E: La clase {0} con anotación no tenía un nombre válido
  </MsgText>
  <Explanation>
    La clase debe especificar un nombre válido en la anotación
  </Explanation>
  <UserResponse>
    Modifique la clase para que especifique un nombre válido en la anotación
  </UserResponse>
</Message>
<Message ID="SRVE8005E" severity="E" prefix="yes">
  <MsgText pgmKey="add.filter.mapping.to.null.url.patterns" varFormat="Java">
    add.filter.mapping.to.null.url.patterns=SRVE8005E: Se ha intentado añadir una correlación de filtros con el parámetro de patrón URL del argumento variable establecido en nulo.
  </MsgText>
  <Explanation>
    El parámetro de patrón URL del argumento variable pasado a FilterRegistration.addMappingForUrlPatterns no debe ser nulo ni debe estar vacío.
  </Explanation>
  <UserResponse>
    Modifique el código de aplicación de acuerdo con la explicación del error.
  </UserResponse>
</Message>
<Message ID="SRVE0300E" severity="E" prefix="yes">
  <MsgText pgmKey="mapping.already.exists" varFormat="Java">
    mapping.already.exists=SRVE0300E: Ya existe la correlación [{0}][{1}]: {2}
  </MsgText>
  <Explanation>
    La vía de acceso deseada ya está asignada en una correlación de servlets. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Cambie o elimine la correlación desde la configuración de la aplicación.
  </UserResponse>
</Message>
<Message ID="SRVE0316W" severity="W" prefix="yes">
  <MsgText pgmKey="request.matches.context.root" varFormat="Java">
    request.matches.context.root=SRVE0316W: La solicitud coincide con la raíz de contexto [{0}] bajo el alias de host virtual de [{1}].
  </MsgText>
  <Explanation>
    La vía de acceso de solicitud coincide con una raíz de contexto en un host virtual no coincidente.
  </Explanation>
  <UserResponse>
    Compruebe si debe añadirse un alias de host virtual.
  </UserResponse>
</Message>
<Message ID="SRVE0210I" severity="I" prefix="yes">
  <MsgText pgmKey="be.debugged.by.recompiling.the.servlet.using.only.the.classes.in.the.application&apos;s.runtime.classpath" varFormat="Java">
    be.debugged.by.recompiling.the.servlet.using.only.the.classes.in.the.application&apos;s.runtime.classpath=SRVE0210I: Este problema puede ser depurado recompilando el servlet utilizando únicamente las clases de la vía de acceso de clases de tiempo de ejecución de la aplicación
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Recompile el servlet utilizando únicamente las clases de la vía de acceso de clases de tiempo de ejecución de la aplicación.
  </UserResponse>
</Message>
<Message ID="SRVE0206E" severity="E" prefix="yes">
  <MsgText pgmKey="This.error.implies.servlet.was.originally.compiled.with.classes.which.cannot.be.located.by.server" varFormat="Java">
    This.error.implies.servlet.was.originally.compiled.with.classes.which.cannot.be.located.by.server=SRVE0206E: Este error normalmente indica que el servlet se ha compilado en un principio con clases que el servidor no puede encontrar.
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Vuelva a compilar el servlet con todos los componentes necesarios ubicados en la vía de acceso de clases.
  </UserResponse>
</Message>
<Message ID="SRVE8055E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.while.flushing.last.response" varFormat="Java">
    Error.while.flushing.last.response=SRVE8055E: Se ha producido una excepción inesperada al vaciar el resto de los datos de respuesta.
  </MsgText>
  <Explanation>
    Se ha producido una excepción inesperada al vaciar el resto de los datos de respuesta.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8004E" severity="E" prefix="yes">
  <MsgText pgmKey="add.servlet.mapping.to.empty.url.patterns" varFormat="Java">
    add.servlet.mapping.to.empty.url.patterns=SRVE8004E: Se ha intentado añadir una correlación de servlets con un parámetro de patrón URL de argumento variable vacío.
  </MsgText>
  <Explanation>
    El parámetro de patrón URL del argumento variable pasado a ServletRegistration.addMapping no debe ser nulo ni debe estar vacío.
  </Explanation>
  <UserResponse>
    Modifique el código de aplicación de acuerdo con la explicación del error.
  </UserResponse>
</Message>
<Message ID="SRVE8025E" severity="E" prefix="yes">
  <MsgText pgmKey="error.occurred.during.async.servlet.handling" varFormat="Java">
    error.occurred.during.async.servlet.handling=SRVE8025E: Se ha producido un error o un tiempo de espera excedido al realizar el proceso del servlet asíncrono.
  </MsgText>
  <Explanation>
    Un error o tiempo de espera excedido producto de un dispatch asíncrono invocará todos los escuchas asíncronos registrados y, a continuación, al mecanismo de la página de errores si no se asigna ni se completa ningún escucha.
  </Explanation>
  <UserResponse>
    Arregle la causa del error buscando la excepción y modificando el código de aplicación o asegúrese de proporcionar un AsyncListener o una página de errores para manejar la causa del error de forma satisfactoria.
  </UserResponse>
</Message>
<Message ID="SRVE8013E" severity="E" prefix="yes">
  <MsgText pgmKey="Invalid.Listener" varFormat="Java">
    Invalid.Listener=SRVE8013E: No hay ningún escucha válido que deba añadirse mediante programación
  </MsgText>
  <Explanation>
    El escucha que se está añadiendo mediante programación no implementa un tipo de escucha válido
  </Explanation>
  <UserResponse>
    Modifique el código de aplicación para añadir un escucha del tipo aceptado
  </UserResponse>
</Message>
<Message ID="SRVE0189E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.occurred.while.finishing.request" varFormat="Java">
    Error.occurred.while.finishing.request=SRVE0189E: Se ha producido un error al finalizar la petición
  </MsgText>
  <Explanation>
    Este error se produce cuando se está completando la solicitud. Puede haber sucedido debido a un error de comunicación con el servidor de aplicaciones pero no debería haber afectado al proceso de la solicitud. Es un error del sistema.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8500W" severity="W" prefix="yes">
  <MsgText pgmKey="servlet.with.same.name.already.exists" varFormat="Java">
    servlet.with.same.name.already.exists=SRVE8500W: El servlet denominado {0} no se puede añadir dinámicamente porque ya existe una configuración de servlet con el mismo nombre.
  </MsgText>
  <Explanation>
    Ya existe un servlet con el mismo nombre en esta aplicación.
  </Explanation>
  <UserResponse>
    No es necesaria ninguna acción.  Compruebe los conflictos en el descriptor de despliegue, anotaciones y código de origen que puedan añadir dinámicamente esta configuración de servlet.
  </UserResponse>
</Message>
<Message ID="SRVE0296E" severity="E" prefix="yes">
  <MsgText pgmKey="log.servlet.message.with.throwable" varFormat="Java">
    log.servlet.message.with.throwable=SRVE0296E: [{0}][{1}][Servlet.LOG]:.{2}:.{3}
  </MsgText>
  <Explanation>
    Mensaje informativo y excepción anotados cronológicamente para un servlet mediante el uso de ServletContext.
  </Explanation>
  <UserResponse>
    Revise el mensaje y la excepción para determinar si es necesaria alguna acción. Puede haber información adicional en las anotaciones cronológicas del servidor.
  </UserResponse>
</Message>
<Message ID="SRVE8089E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.creating.stats.group" varFormat="Java">
    Error.creating.stats.group=SRVE8089E: Se ha producido un error al crear el grupo de estadísticas de Performance Monitoring Infrastructure (PMI).
  </MsgText>
  <Explanation>
    Se ha producido un error al crear el grupo de estadísticas de Performance Monitoring Infrastructure (PMI).
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0266E" severity="E" prefix="yes">
  <MsgText pgmKey="error.while.initializing.servlets" varFormat="Java">
    error.while.initializing.servlets=SRVE0266E: Se ha producido un error al inicializar los servlets: {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0219I" severity="I" prefix="yes">
  <MsgText pgmKey="No.Error.to.Report" varFormat="Java">
    No.Error.to.Report=SRVE0219I: No hay ningún error que notificar
  </MsgText>
  <Explanation>
    Se ha invocado DefaultErrorReport pero no se ha encontrado ningún error. Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise la aplicación para determinar la causa del problema.
  </UserResponse>
</Message>
<Message ID="SRVE0298E" severity="E" prefix="yes">
  <MsgText pgmKey="exception.occured.while.creating.wrapper.for.servlet" varFormat="Java">
    exception.occured.while.creating.wrapper.for.servlet=SRVE0298E: Se ha producido una excepción al crear el derivador para el servlet [{0}]: {1}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE8066E" severity="E" prefix="yes">
  <MsgText pgmKey="Failed.to.add.WebContainerService" varFormat="Java">
    Failed.to.add.WebContainerService=SRVE8066E: Se ha producido una excepción inesperada al añadir el WebContainerService.
  </MsgText>
  <Explanation>
    Se ha producido una excepción inesperada al añadir el WebContainerService.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8085E" severity="E" prefix="yes">
  <MsgText pgmKey="handleRequest.threw.IOException" varFormat="Java">
    handleRequest.threw.IOException=SRVE8085E: Se ha emitido una IOException desde el método WebContainer&apos;s handleRequest.
  </MsgText>
  <Explanation>
    Se ha emitido una IOException desde el método WebContainer&apos;s handleRequest.
  </Explanation>
  <UserResponse>
    Recopile registros de servidor y póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0156E" severity="E" prefix="yes">
  <MsgText pgmKey="failed.to.load.converter.properties" varFormat="Java">
    failed.to.load.converter.properties=SRVE0156E: No se ha podido cargar el archivo converter.properties {0}
  </MsgText>
  <Explanation>
    No se ha podido cargar el archivo especificado. Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Asegúrese de que el archivo especificado exista y de que tenga el formato correcto de acuerdo con java.utils.Properties.
  </UserResponse>
</Message>
<Message ID="SRVE8093W" severity="W" prefix="yes">
  <MsgText pgmKey="servlet.classname.is.null" varFormat="Java">
    servlet.classname.is.null=SRVE8093W: El nombre de clase del servlet es nulo para el servlet denominado [{0}] por lo que no se puede procesar la solicitud.
  </MsgText>
  <Explanation>
    El nombre de clase del servlet es nulo por lo que no se puede procesar la solicitud.
  </Explanation>
  <UserResponse>
    Compruebe que se facilite una clase de servlet válida para el servlet.
  </UserResponse>
</Message>
<Message ID="SRVE8098W" severity="W" prefix="yes">
  <MsgText pgmKey="null.url.pattern.for.servlet.mapping" varFormat="Java">
    null.url.pattern.for.servlet.mapping=SRVE8098W: No se ha podido insertar la correlación para el servlet denominado [{0}]. El patrón de URL es nulo.
  </MsgText>
  <Explanation>
    Una correlación de servlets está incompleta sin un patrón de URL con el que deba correlacionarse.
  </Explanation>
  <UserResponse>
    Añada un patrón de URL a la correlación de servlets.
  </UserResponse>
</Message>
<Message ID="SRVE8033W" severity="W" prefix="yes">
  <MsgText pgmKey="exception.occurred.while.initializing.ServletContainerInitializers" varFormat="Java">
    exception.occurred.while.initializing.ServletContainerInitializers=SRVE8033W: Se ha producido un problema al inicializar ServletContainerInitializers de la aplicación.
  </MsgText>
  <Explanation>
    Se ha producido un problema al inicializar ServletContainerInitializers de la aplicación.
  </Explanation>
  <UserResponse>
    Verifique que el ServletContainerInitializer esté bien configurado.
  </UserResponse>
</Message>
<Message ID="SRVE0321E" severity="E" prefix="yes">
  <MsgText pgmKey="init.exception.thrown.by.filter.at.startup" varFormat="Java">
    init.exception.thrown.by.filter.at.startup=SRVE0321E: El filtro [{0}] no se ha cargado durante el inicio.
  </MsgText>
  <Explanation>
    El filtro no se ha cargado durante el inicio.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema.
  </UserResponse>
</Message>
<Message ID="SRVE8041E" severity="E" prefix="yes">
  <MsgText pgmKey="trying.to.call.dispatch.twice.for.the.same.async.operation" varFormat="Java">
    trying.to.call.dispatch.twice.for.the.same.async.operation=SRVE8041E: Se está intentando llamar a dispatch por duplicado para la misma operación asíncrona
  </MsgText>
  <Explanation>
    La especificación del servlet indica que sólo se puede llamar a dispatch una vez por llamada a startAsync
  </Explanation>
  <UserResponse>
    Elimine la llamada adicional a dispatch.
  </UserResponse>
</Message>
<Message ID="SRVE0324W" severity="W" prefix="yes">
  <MsgText pgmKey="ARD.not.enabled.request.on.chains.not.supported" varFormat="Java">
    ARD.not.enabled.request.on.chains.not.supported=SRVE0324W: El Asynchronous Request Dispatcher no está habilitado. Por lo tanto, es posible que las cadenas HttpQueueInboundDefault y HttpQueueInboundDefaultSecure no gestionen las solicitudes correctamente.
  </MsgText>
  <Explanation>
    Las solicitudes están en la cadena HttpQueueInboundDefault o HttpQueueInboundDefaultSecure, pero el Asynchronous Request Dispatcher no está habilitado en el nivel de servidor. Por lo tanto, es posible que las cadenas HttpQueueInboundDefault y HttpQueueInboundDefaultSecure no gestionen estas solicitudes correctamente.
  </Explanation>
  <UserResponse>
    Habilite el Asynchronous Request Dispatcher en el nivel de servidor, o habilite la cadena WCInboundDefault o WCInboundDefaultSecure y asegúrese de que esté en ejecución.
  </UserResponse>
</Message>
<Message ID="SRVE0035E" severity="E" prefix="yes">
  <MsgText pgmKey="WebAppSecurityCollaborator" varFormat="Java">
    WebAppSecurityCollaborator=SRVE0035E: WebAppSecurityCollaborator
  </MsgText>
  <Explanation>
    WebAppSecurityCollaborator. Es un error del sistema.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema. Si no existen mensajes relacionados o no ayudan a resolver el problema, póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8034E" severity="E" prefix="yes">
  <MsgText pgmKey="IOException.loading.servlet" varFormat="Java">
    IOException.loading.servlet=SRVE8034E: Servlet [{0}]: {1} no se ha podido cargar debido a una IOException (excepción de entrada/salida).
  </MsgText>
  <Explanation>
    Una IOException puede producirse por numerosas razones relacionados con la entrada/salida.
  </Explanation>
  <UserResponse>
    Compruebe si se ha eliminado la clase.
  </UserResponse>
</Message>
<Message ID="SRVE0224I" severity="I" prefix="yes">
  <MsgText pgmKey="Target.Servlet" varFormat="Java">
    Target.Servlet=SRVE0224I: Servlet de destino:
  </MsgText>
  <Explanation>
    Ninguna.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0261E" severity="E" prefix="yes">
  <MsgText pgmKey="annotation.helper.not.defined" varFormat="Java">
    annotation.helper.not.defined=SRVE0261E: El ayudante de anotación {0} no se ha definido.
  </MsgText>
  <Explanation>
    El ayudante de anotación especificado no se ha definido. Es un error del sistema.
  </Explanation>
  <UserResponse>
    Compruebe la definición del ayudante de anotación especificado y añádala si es necesario.
  </UserResponse>
</Message>
<Message ID="SRVE0068E" severity="E" prefix="yes">
  <MsgText pgmKey="Uncaught.service().exception.thrown.by.servlet" varFormat="Java">
    Uncaught.service().exception.thrown.by.servlet=SRVE0068E: Ha sido generada una excepción por parte de uno de los métodos de servicio del servlet [{0}] en la aplicación [{1}]. Excepción creada: [{2}]
  </MsgText>
  <Explanation>
    Ha sido generada una excepción por parte de uno de los métodos de servicio del servlet. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Examine el mensaje de excepción, el código fuente del servlet y las anotaciones cronológicas del servidor para determinar la causa raíz del problema.
  </UserResponse>
</Message>
<Message ID="SRVE8062E" severity="E" prefix="yes">
  <MsgText pgmKey="exception.initializing.security.collaborator" varFormat="Java">
    exception.initializing.security.collaborator=SRVE8062E: Se ha producido una excepción inesperada al inicializar el colaborador de seguridad.
  </MsgText>
  <Explanation>
    Se ha producido una excepción inesperada al inicializar el colaborador de seguridad.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0269W" severity="W" prefix="yes">
  <MsgText pgmKey="no.jsp.extension.handler.found" varFormat="Java">
    no.jsp.extension.handler.found=SRVE0269W: No se ha encontrado ningún procesador de extensiones para manejar JSP
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0203E" severity="E" prefix="yes">
  <MsgText pgmKey="Servlet.was.found.but.is.missing.another.required.class" varFormat="Java">
    Servlet.was.found.but.is.missing.another.required.class=SRVE0203E: Servlet [{0}]: se ha encontrado {1}, pero le falta otra clase necesaria.
  </MsgText>
  <Explanation>
    No se ha encontrado una clase necesaria. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Asegúrese de que todas las clases necesarias estén ubicadas en carpetas/jars que representen la vía de acceso de clases
  </UserResponse>
</Message>
<Message ID="SRVE0099E" severity="E" prefix="yes">
  <MsgText pgmKey="Object.Pool.Exception.Class.could.not.be.instantiated.[{0}]" varFormat="Java">
    Object.Pool.Exception.Class.could.not.be.instantiated.[{0}]=SRVE0099E: Excepción de agrupación de objetos: No se ha podido instanciar la clase [{0}].
  </MsgText>
  <Explanation>
    No se ha podido crear la instancia de una clase para que la utilice la agrupación de objetos. Es un error del sistema.
  </Explanation>
  <UserResponse>
    Asegúrese de que la clase exista y que sea visible en la vía de acceso de clases de la aplicación.
  </UserResponse>
</Message>
<Message ID="SRVE8028W" severity="W" prefix="yes">
  <MsgText pgmKey="default.extension.exception.adding.mapping.target" varFormat="Java">
    default.extension.exception.adding.mapping.target=SRVE8028W: Se ha producido una excepción al añadir un destino de correlación al procesador de extensiones predeterminado.
  </MsgText>
  <Explanation>
    Cuando se hacen dos solicitudes por primera vez al mismo recurso, puede producirse una condición de actualización al intentar añadir la correlación a la aplicación web.
  </Explanation>
  <UserResponse>
    Normalmente se trata de una condición que se puede ignorar. Sólo notifíquelo si se producen problemas al servir solicitudes a los recursos en cuestión.
  </UserResponse>
</Message>
<Message ID="SRVE0202E" severity="E" prefix="yes">
  <MsgText pgmKey="Servlet.found.but.corrupt" varFormat="Java">
    Servlet.found.but.corrupt=SRVE0202E: Servlet [{0}]: Se ha encontrado {1}, pero está corrompida:
  </MsgText>
  <Explanation>
    La clase de servlet ha resultado estar corrompida. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Vuelva a compilar la clase e inténtelo de nuevo.
  </UserResponse>
</Message>
<Message ID="SRVE0105E" severity="E" prefix="yes">
  <MsgText pgmKey="Session.releaseSession().threw.an.unexpected.exception" varFormat="Java">
    Session.releaseSession().threw.an.unexpected.exception=SRVE0105E: Se ha producido una excepción en Session.releaseSession()
  </MsgText>
  <Explanation>
    Problema al liberar una sesión.
  </Explanation>
  <UserResponse>
    Examine las anotaciones cronológicas del servidor para determinar la causa raíz del problema. Si no existen mensajes relacionados o no ayudan a resolver el problema, póngase en contacto con el servicio de soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8106W" severity="W" prefix="yes">
  <MsgText pgmKey="exception.invoking.async.listener" varFormat="Java">
    exception.invoking.async.listener=SRVE8106W: Se ha producido una excepción al invocar las escuchas asíncronas.
  </MsgText>
  <Explanation>
    Se ha producido una excepción al invocar las escuchas asíncronas.
  </Explanation>
  <UserResponse>
    Consulte la causa raíz.
  </UserResponse>
</Message>
<Message ID="SRVE8107W" severity="W" prefix="yes">
  <MsgText pgmKey="trying.to.schedule.timeout.without.cancelling.previous.timeout" varFormat="Java">
    trying.to.schedule.timeout.without.cancelling.previous.timeout=SRVE8107W: AsyncContext ha intentado programar una hebra de tiempo de espera sin cancelar el tiempo de espera anterior.
  </MsgText>
  <Explanation>
    Se trata de un error interno inesperado.
  </Explanation>
  <UserResponse>
    Recopile el rastreo del contenedor web y póngase en contacto con el soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0169I" severity="I" prefix="yes">
  <MsgText pgmKey="loading.web.module" varFormat="Java">
    loading.web.module=SRVE0169I: Cargando módulo web: {0}.
  </MsgText>
  <Explanation>
    El módulo Web se está iniciando y está disponible para el servicio.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8051E" severity="E" prefix="yes">
  <MsgText pgmKey="Exception.occured.during.servlet.unload" varFormat="Java">
    Exception.occured.during.servlet.unload=SRVE8051E: Se ha producido un error de servidor interno al descargar el servlet.
  </MsgText>
  <Explanation>
    Se ha producido un error de servidor interno al descargar el servlet.
  </Explanation>
  <UserResponse>
    Consulte la causa raíz de la excepción.
  </UserResponse>
</Message>
<Message ID="SRVE0305W" severity="W" prefix="yes">
  <MsgText pgmKey="Object.Pool.Disabled" varFormat="Java">
    Object.Pool.Disabled=SRVE0305W: ObjectPoolService no está disponible. Se ha inhabilitado la agrupación de SRTConnectionContext.
  </MsgText>
  <Explanation>
    El servicio de agrupaciones de objeto mejora el rendimiento del contenedor web, pero se ha inhabilitado. Es un error del sistema.
  </Explanation>
  <UserResponse>
    Es recomendable habilitar el servicio de agrupación de objetos.
  </UserResponse>
</Message>
<Message ID="SRVE0258W" severity="W" prefix="yes">
  <MsgText pgmKey="property.not.applicable" varFormat="Java">
    property.not.applicable=SRVE0258W: Ya no se puede aplicar el atributo {0} bajo el transporte.
  </MsgText>
  <Explanation>
    Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Configure esta propiedad bajo el Servicio de anotación cronológica (consulte el Information Center para obtener más detalles sobre el Registrador de acceso HTTP (HTTP Access Logger).
  </UserResponse>
</Message>
<Message ID="SRVE8083E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.occurred.while.registering.WebModule.MBean" varFormat="Java">
    Error.occurred.while.registering.WebModule.MBean=SRVE8083E: Se ha producido un error de servidor interno inesperado al registrar el MBean gestionado del módulo web.
  </MsgText>
  <Explanation>
    Se ha producido un error de servidor interno inesperado al registrar el MBean gestionado del módulo web.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0240I" severity="I" prefix="yes">
  <MsgText pgmKey="ExtensionFactory.[{0}].associated.with.patterns.[{1}]" varFormat="Java">
    ExtensionFactory.[{0}].associated.with.patterns.[{1}]=SRVE0240I: Extension Factory [{0}] se ha asociado con patrones [{1}].
  </MsgText>
  <Explanation>
    Ninguna.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0232E" severity="E" prefix="yes">
  <MsgText pgmKey="Engine.Exception.[{0}]" varFormat="Java">
    Engine.Exception.[{0}]=SRVE0232E: Error interno del servidor. &lt;br&gt; Mensaje de excepción: [{0}]
  </MsgText>
  <Explanation>
    Se ha producido una excepción inesperada y se han facilitado detalles en la página de errores resultante. Es un error del sistema.
  </Explanation>
  <UserResponse>
    Compruebe los registros System.out y System.err para obtener más información sobre la excepción generada.
  </UserResponse>
</Message>
<Message ID="SRVE0273E" severity="E" prefix="yes">
  <MsgText pgmKey="error.while.initializing.jsp.as.servlet" varFormat="Java">
    error.while.initializing.jsp.as.servlet=SRVE0273E: Se ha producido un error mientras se inicializaba JSP como servlet [{0}] en la aplicación [{1}]: {1}
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0314E" severity="E" prefix="yes">
  <MsgText pgmKey="failed.to.remove.webmodule" varFormat="Java">
    failed.to.remove.webmodule=SRVE0314E: No se ha podido eliminar el módulo web {0}: {1}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0326E" severity="E" prefix="yes">
  <MsgText pgmKey="Exceeding.maximum.hash.collisions" varFormat="Java">
    Exceeding.maximum.hash.collisions=SRVE0326E: Se ha superado el número máximo de conflictos de código hash de clave de parámetro permitidos por solicitud {0}.
  </MsgText>
  <Explanation>
    El número de conflictos de código hash de clave de parámetro supera el número máximo de conflictos de código hash de clave de parámetro permitidos por solicitud de entrada. Es posible que las claves de parámetro que generan códigos hash duplicados se hayan añadido malintencionadamente a la solicitud, provocando un gran número de conflictos de código hash de clave de parámetro.
  </Explanation>
  <UserResponse>
    Asegúrese de que todos los parámetros de la solicitud de entrada sean válidos.
  </UserResponse>
</Message>
<Message ID="SRVE0323I" severity="I" prefix="yes">
  <MsgText pgmKey="ARD.not.enabled.these.chains.not.supported" varFormat="Java">
    ARD.not.enabled.these.chains.not.supported=SRVE0323I: Asynchronous Request Dispatcher no está habilitado. Es posible que las solicitudes enviadas a través de la cadena HttpQueueInboundDefault o HttpQueueInboundDefaultSecure no se hayan gestionado correctamente.
  </MsgText>
  <Explanation>
    Es posible que las solicitudes a través de la cadena HttpQueueInboundDefault o HttpQueueInboundDefaultSecure no se hayan gestionado correctamente si el Asynchronous Request Dispatcher no está habilitado en el nivel de servidor.
  </Explanation>
  <UserResponse>
    Si el Asynchronous Request Dispatcher no está habilitado en el nivel de servidor, asegúrese de que la cadena WCInboundDefault o WCInboundDefaultSecure esté habilitada y en ejecución. Si ninguna de estas cadenas está en ejecución, debe habilitar el Asynchronous Request Dispatcher, si todavía no está habilitado porque el servidor enviará las solicitudes a través de la cadena HttpQueueInboundDefault o HttpQueueInboundDefaultSecure.
  </UserResponse>
</Message>
<Message ID="SRVE8099W" severity="W" prefix="yes">
  <MsgText pgmKey="stub.restart.web.application" varFormat="Java">
    stub.restart.web.application=SRVE8099W: El método restartWebApplication ya no hace nada. Utilice el bean gestionado ApplicationManager para iniciar y reiniciar una aplicación.
  </MsgText>
  <Explanation>
    El método restartWebApplication ya no hace nada. Utilice el bean gestionado ApplicationManager para iniciar y reiniciar una aplicación.
  </Explanation>
  <UserResponse>
    El método restartWebApplication ya no hace nada. Utilice el bean gestionado ApplicationManager para iniciar y reiniciar una aplicación.
  </UserResponse>
</Message>
<Message ID="SRVE0309E" severity="E" prefix="yes">
  <MsgText pgmKey="servlet.on.blocked.list.{0}" varFormat="Java">
    servlet.on.blocked.list.{0}=SRVE0309E: Servlet en la lista de elementos bloqueados: {0}
  </MsgText>
  <Explanation>
    El servlet solicitado se encuentra en la lista de elementos bloqueados proporcionada por la propiedad personalizada com.ibm.ws.webcontainer.donotservebyclassname. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Si no desea este comportamiento, actualice la propiedad com.ibm.ws.webcontainer.donotservebyclassname.
  </UserResponse>
</Message>
<Message ID="SRVE0190E" severity="E" prefix="yes">
  <MsgText pgmKey="File.not.found" varFormat="Java">
    File.not.found=SRVE0190E: Archivo no encontrado: {0}
  </MsgText>
  <Explanation>
    No se ha encontrado el archivo mencionado. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Asegúrese de que el archivo mencionado esté en su sitio.
  </UserResponse>
</Message>
<Message ID="SRVE8076E" severity="E" prefix="yes">
  <MsgText pgmKey="Null.AsyncRequestDispatcherConfigImpl.object" varFormat="Java">
    Null.AsyncRequestDispatcherConfigImpl.object=SRVE8076E: Objeto AsyncRequestDispatcherConfigImpl nulo.
  </MsgText>
  <Explanation>
    Se ha producido una excepción de configuración inesperada.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8081E" severity="E" prefix="yes">
  <MsgText pgmKey="Error.occurred.while.adding.WebAppCollaborator" varFormat="Java">
    Error.occurred.while.adding.WebAppCollaborator=SRVE8081E: Se ha producido un error de servidor interno inesperado al añadir un colaborador de aplicación web.
  </MsgText>
  <Explanation>
    Se ha producido un error de servidor interno inesperado al añadir un colaborador de aplicación web.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0257W" severity="W" prefix="yes">
  <MsgText pgmKey="property.has.changed" varFormat="Java">
    property.has.changed=SRVE0257W: Se ha cambiado la semántica de la propiedad {0} en el transporte.
  </MsgText>
  <Explanation>
    Ya no hay una hebra unida a una única conexión. Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Consulte el Information Center para obtener más detalles.
  </UserResponse>
</Message>
<Message ID="SRVE0230E" severity="E" prefix="yes">
  <MsgText pgmKey="class.compiled.using.proper.case" varFormat="Java">
    class.compiled.using.proper.case=SRVE0230E: Compruebe que la clase se haya compilado con mayúsculas o minúsculas según corresponda (tal como indica la definición de clase).
  </MsgText>
  <Explanation>
    Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Compruebe que la clase se haya compilado con mayúsculas o minúsculas según corresponda (tal como indica la definición de clase).
  </UserResponse>
</Message>
<Message ID="SRVE8064E" severity="E" prefix="yes">
  <MsgText pgmKey="Could.not.create.LogoutProcessor" varFormat="Java">
    Could.not.create.LogoutProcessor=SRVE8064E: Se ha producido una excepción inesperada al inicializar el procesador de cierre de sesión.
  </MsgText>
  <Explanation>
    Se ha producido una excepción inesperada al inicializar el procesador de cierre de sesión.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0218E" severity="E" prefix="yes">
  <MsgText pgmKey="Forbidden.Web.Security.Exception" varFormat="Java">
    Forbidden.Web.Security.Exception=SRVE0218E: Prohibición: Excepción de seguridad de Web
  </MsgText>
  <Explanation>
    La solicitud ha intentado acceder a un recurso prohibido.
  </Explanation>
  <UserResponse>
    Asegúrese de que la entidad solicitante tenga los privilegios necesarios.
  </UserResponse>
</Message>
<Message ID="SRVE0244I" severity="I" prefix="yes">
  <MsgText pgmKey="reaper.unloading.servlet.[{0}]" varFormat="Java">
    reaper.unloading.servlet.[{0}]=SRVE0244I: Hebra segadora al descargar el servlet: [{0}].
  </MsgText>
  <Explanation>
    La hebra segadora elimina servlets que han estado inactivos superando el límite de inactividad de la segadora.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8030E" severity="E" prefix="yes">
  <MsgText pgmKey="exception.occured.while.adding.servlet" varFormat="Java">
    exception.occured.while.adding.servlet=SRVE8030E: Se ha producido una ServletException al añadir un servlet dinámicamente.
  </MsgText>
  <Explanation>
    Se ha producido una ServletException inesperada añadiendo un servlet dinámicamente. Esto puede suceder, por ejemplo, si los nombres de los beans gestionados de dos servlets entran en colisión.
  </Explanation>
  <UserResponse>
    Revise los registros y póngase en contacto con el soporte de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE8007E" severity="E" prefix="yes">
  <MsgText pgmKey="add.filter.mapping.to.null.servlet.names" varFormat="Java">
    add.filter.mapping.to.null.servlet.names=SRVE8007E: Se ha intentado añadir una correlación de filtros con el parámetro de nombre de servlet de argumento variable establecido en nulo.
  </MsgText>
  <Explanation>
    El parámetro de nombre de servlet de argumento variable pasado a FilterRegistration.addMappingForServletNames no debe ser nulo ni debe estar vacío.
  </Explanation>
  <UserResponse>
    Modifique el código de aplicación de acuerdo con la explicación del error.
  </UserResponse>
</Message>
<Message ID="SRVE8070E" severity="E" prefix="yes">
  <MsgText pgmKey="failed.to.initialize.webcontainer.required.to.start.chains" varFormat="Java">
    failed.to.initialize.webcontainer.required.to.start.chains=SRVE8070E: Se ha producido un error inesperado al iniciar las cadenas de canal del contenedor web.
  </MsgText>
  <Explanation>
    Se ha producido un error inesperado al iniciar las cadenas de canal del contenedor web.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0196E" severity="E" prefix="yes">
  <MsgText pgmKey="Missing.required.initialization.parameter" varFormat="Java">
    Missing.required.initialization.parameter=SRVE0196E: Falta el parámetro de inicialización necesario: {0}
  </MsgText>
  <Explanation>
    El parámetro con el nombre mencionada no tiene ningún valor. Se trata de un parámetro obligatorio. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Proporcione un valor para el parámetro mencionado.
  </UserResponse>
</Message>
<Message ID="SRVE0200E" severity="E" prefix="yes">
  <MsgText pgmKey="Servlet.Could.not.find.required.servlet.class" varFormat="Java">
    Servlet.Could.not.find.required.servlet.class=SRVE0200E: Servlet [{0}]: No se ha encontrado la clase necesaria - {1}
  </MsgText>
  <Explanation>
    El servlet no ha podido encontrar la clase necesaria para servir a la solicitud. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Verifique que exista la clase dentro de la aplicación.
  </UserResponse>
</Message>
<Message ID="SRVE8042E" severity="E" prefix="yes">
  <MsgText pgmKey="reference.context.not.set" varFormat="Java">
    reference.context.not.set=SRVE8042E: Un error interno ha causado que el contexto de referencia que habilita la inyección no se inicializara de forma correcta.
  </MsgText>
  <Explanation>
    Un error interno ha causado que el contexto de referencia que habilita la inyección no se inicializara de forma correcta.
  </Explanation>
  <UserResponse>
    Mire los errores anteriores en los registros pues es posible que hayan podido causar que la inicialización fallara.
  </UserResponse>
</Message>
<Message ID="SRVE0235E" severity="E" prefix="yes">
  <MsgText pgmKey="[{0}].reported.an.error" varFormat="Java">
    [{0}].reported.an.error=SRVE0235E: [{0}] ha notificado un error
  </MsgText>
  <Explanation>
    Consulte el error notificado.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0290E" severity="E" prefix="yes">
  <MsgText pgmKey="unable.to.use.specified.temp.directory" varFormat="Java">
    unable.to.use.specified.temp.directory=SRVE0290E: No se puede utilizar el directorio temporal especificado: {0} leer:{1} escribir:{2}
  </MsgText>
  <Explanation>
    Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Compruebe el estado del directorio, incluyendo la existencia, los privilegios de acceso y el espacio disponible.
  </UserResponse>
</Message>
<Message ID="SRVE0161I" severity="I" prefix="yes">
  <MsgText pgmKey="web.container.copyright" varFormat="Java">
    web.container.copyright=SRVE0161I: IBM WebSphere Application Server - Contenedor web.  Copyright IBM Corp. 1998-2012
  </MsgText>
  <Explanation>
    Identificador del servidor de aplicaciones.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0058E" severity="E" prefix="yes">
  <MsgText pgmKey="Uncaught.destroy().exception.thrown.by.servlet" varFormat="Java">
    Uncaught.destroy().exception.thrown.by.servlet=SRVE0058E: No se ha producido la excepción destroy() creada por el servlet {0} en la aplicación {1}: {2}
  </MsgText>
  <Explanation>
    El método destroy() servlet ha lanzado una excepción. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    Examine el mensaje de excepción, el código fuente del servlet y las anotaciones cronológicas del servidor para determinar la causa raíz del problema.
  </UserResponse>
</Message>
<Message ID="SRVE0186E" severity="E" prefix="yes">
  <MsgText pgmKey="Cannot.set.buffer.size.after.data" varFormat="Java">
    Cannot.set.buffer.size.after.data=SRVE0186E: No se puede establecer el tamaño de almacenamiento intermedio una vez grabados los datos en la corriente de datos
  </MsgText>
  <Explanation>
    Algunos datos ya se han grabado en la secuencia y, debido al tamaño del almacenamiento intermedio, ya no se pueden controlar. Se trata de un error de aplicación.
  </Explanation>
  <UserResponse>
    O bien establezca el tamaño del almacenamiento intermedio antes de grabar datos o bien restablezca el almacenamiento intermedio antes de fijar el tamaño.
  </UserResponse>
</Message>
<Message ID="SRVE0205E" severity="E" prefix="yes">
  <MsgText pgmKey="host.on.port.has.not.been.defined" varFormat="Java">
    host.on.port.has.not.been.defined=SRVE0205E: El host {0} en el puerto {1} no se ha definido.
  </MsgText>
  <Explanation>
    No se ha podido ubicar el host virtual definido para manejar este URI de solicitud. Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Compruebe la configuración del host virtual en el archivo virtualhosts.xml o mediante la consola de administración.
  </UserResponse>
</Message>
<Message ID="SRVE8092W" severity="W" prefix="yes">
  <MsgText pgmKey="exception.invoking.asnyc.error.mechanism" varFormat="Java">
    exception.invoking.asnyc.error.mechanism=SRVE8092W: Se ha producido una excepción al invocar el mecanismo de errores asíncronos.
  </MsgText>
  <Explanation>
    Se ha producido una excepción al invocar el mecanismo de errores asíncronos.
  </Explanation>
  <UserResponse>
    Consulte la causa raíz.
  </UserResponse>
</Message>
<Message ID="SRVE0061E" severity="E" prefix="yes">
  <MsgText pgmKey="Illegal.Argument:.Invalid.Directory.Specified:.{0}" varFormat="Java">
    Illegal.Argument:.Invalid.Directory.Specified:.{0}=SRVE0061E: Excepción de argumento no permitido: El directorio especificado no es válido: {0}
  </MsgText>
  <Explanation>
    El directorio especificado no es válido.
  </Explanation>
  <UserResponse>
    Compruebe si hay algún error en la especificación del directorio.
  </UserResponse>
</Message>
<Message ID="SRVE0284E" severity="E" prefix="yes">
  <MsgText pgmKey="exception.caught.in.notifyServletContextCreated" varFormat="Java">
    exception.caught.in.notifyServletContextCreated=SRVE0284E: Se ha detectado una excepción en notifyServletContextCreated: {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE8043E" severity="E" prefix="yes">
  <MsgText pgmKey="ioexception.searching.jar.for.resource" varFormat="Java">
    ioexception.searching.jar.for.resource=SRVE8043E: Se ha producido una IOException al buscar jarFile [{1}] para fileName [{2}] {0}
  </MsgText>
  <Explanation>
    Las IOExceptions pueden producirse al buscar archivos JAR si el archivo se cambia o se cierra durante el cruce.
  </Explanation>
  <UserResponse>
    Verifique que el archivo JAR sea válido y que no sea actualizado durante el cruce.
  </UserResponse>
</Message>
<Message ID="SRVE0155E" severity="E" prefix="yes">
  <MsgText pgmKey="failed.to.load.encoding.properties" varFormat="Java">
    failed.to.load.encoding.properties=SRVE0155E: No se ha podido cargar el archivo encoding.properties {0}
  </MsgText>
  <Explanation>
    No se ha podido cargar el archivo especificado. Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Asegúrese de que el archivo especificado exista y de que tenga el formato correcto de acuerdo con java.utils.Properties.
  </UserResponse>
</Message>
<Message ID="SRVE0248I" severity="I" prefix="yes">
  <MsgText pgmKey="waiting.to.destroy.filter.[{0}]" varFormat="Java">
    waiting.to.destroy.filter.[{0}]=SRVE0248I: Actualmente se están procesando las solicitudes. Espere hasta 60 segundos antes de forzar la destrucción del filtro.
  </MsgText>
  <Explanation>
    Ninguna.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0204E" severity="E" prefix="yes">
  <MsgText pgmKey="host.has.not.been.defined" varFormat="Java">
    host.has.not.been.defined=SRVE0204E: El host {0} no se ha definido
  </MsgText>
  <Explanation>
    No se ha encontrado el host virtual. Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Compruebe la configuración del host virtual en el archivo virtualhosts.xml o mediante la consola de administración.
  </UserResponse>
</Message>
<Message ID="SRVE8110E" severity="E" prefix="yes">
  <MsgText pgmKey="exception.occured.while.processing.ServletContainerInitializer.initializeExtensionPointSCIs" varFormat="Java">
    exception.occured.while.processing.ServletContainerInitializer.initializeExtensionPointSCIs=SRVE8110E: No se ha podido inicializar ServletContainerInitializer: Classname: [{0}].
  </MsgText>
  <Explanation>
    El ServletContainerInitializer especificado por el punto de extensión no se ha podido inicializar.
  </Explanation>
  <UserResponse>
    Compruebe la clase especificada en el punto de extensión.
  </UserResponse>
</Message>
<Message ID="SRVE0281E" severity="E" prefix="yes">
  <MsgText pgmKey="request.processor.already.present.for.mapping" varFormat="Java">
    request.processor.already.present.for.mapping=SRVE0281E: El procesador de solicitudes ya está presente en la correlación: {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0265E" severity="E" prefix="yes">
  <MsgText pgmKey="error.notifying.listeners.of.WebApp.start" varFormat="Java">
    error.notifying.listeners.of.WebApp.start=SRVE0265E: Se ha producido un error al notificar a los escuchas el inicio de la aplicación web: {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Revise los registros.
  </UserResponse>
</Message>
<Message ID="SRVE0260E" severity="E" prefix="yes">
  <MsgText pgmKey="cannot.use.error.page" varFormat="Java">
    cannot.use.error.page=SRVE0260E: El servidor no puede utilizar la página de errores para que su aplicación maneje la excepción original abajo impresa.
  </MsgText>
  <Explanation>
    Ninguna.
  </Explanation>
  <UserResponse>
    Consulte Excepción de página de errores para obtener una descripción del problema con la página de errores especificada.
  </UserResponse>
</Message>
<Message ID="SRVE0223I" severity="I" prefix="yes">
  <MsgText pgmKey="StackTrace" varFormat="Java">
    StackTrace=SRVE0223I: StackTrace:
  </MsgText>
  <Explanation>
    Ninguna.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0211E" severity="E" prefix="yes">
  <MsgText pgmKey="class.{0}.could.not.be.instantiated" varFormat="Java">
    class.{0}.could.not.be.instantiated=SRVE0211E: La clase {0} no se ha podido instanciar
  </MsgText>
  <Explanation>
    Excepción interna.
  </Explanation>
  <UserResponse>
    Si el problema persiste, puede haber información adicional disponible si busca el ID del mensaje en los sitios Web siguientes: página de soporte de IBM WebSphere Application Server: http://www.ibm.com/software/webservers/appserv/was/support/   página de soporte de IBM WebSphere Application Server for z/OS: http://www.ibm.com/software/webservers/appserv/zos_os390/support/
  </UserResponse>
</Message>
<Message ID="SRVE0021I" severity="I" prefix="yes">
  <MsgText pgmKey="Servlet.unloaded:.{0}" varFormat="Java">
    Servlet.unloaded:.{0}=SRVE0021I: Servlet no cargado: {0}
  </MsgText>
  <Explanation>
    El servlet especificado no se ha cargado después de haber llamado al método destroy (destrucción).
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8079E" severity="E" prefix="yes">
  <MsgText pgmKey="Null.filter.config.object" varFormat="Java">
    Null.filter.config.object=SRVE8079E: Objeto de configuración de filtro nulo.
  </MsgText>
  <Explanation>
    Se ha producido una excepción de configuración inesperada.
  </Explanation>
  <UserResponse>
    Recopile las anotaciones cronológicas del servidor y póngase en contacto con el servicio de IBM.
  </UserResponse>
</Message>
<Message ID="SRVE0306W" severity="W" prefix="yes">
  <MsgText pgmKey="ARD.Not.Enabled" varFormat="Java">
    ARD.Not.Enabled=SRVE0306W: ARD no está habilitado por lo que una inclusión asíncrona o llamada insertFragment será síncrona.
  </MsgText>
  <Explanation>
    ARD no está bien habilitado. Se requiere una acción del sistema.
  </Explanation>
  <UserResponse>
    Asegúrese de que ARD está habilitado en los valores del contenedor web y se ha especificado un tipo de envío CLIENT_SIDE o SERVER_SIDE en los valores de la aplicación.
  </UserResponse>
</Message>
<Message ID="SRVE0325E" severity="E" prefix="yes">
  <MsgText pgmKey="Exceeding.maximum.parameters" varFormat="Java">
    Exceeding.maximum.parameters=SRVE0325E: Se ha superado el número máximo de parámetros permitidos por solicitud {0}, actual {1}, no se pueden añadir más.
  </MsgText>
  <Explanation>
    El número de parámetros contenidos en la solicitud de entrada supera el número máximo de parámetros que pueden incluirse en una solicitud de entrada.
  </Explanation>
  <UserResponse>
    Asegúrese de que todos los parámetros de la solicitud de entrada sean válidos y de que el número total de parámetros no supere el máximo permitido por solicitud.
  </UserResponse>
</Message>
<Message ID="SRVE8044E" severity="E" prefix="yes">
  <MsgText pgmKey="programmatic.sessions.already.been.initialized" varFormat="Java">
    programmatic.sessions.already.been.initialized=SRVE8044E: El contexto ya se ha inicializado.  No se permiten actualizaciones en la configuración de la sesión.
  </MsgText>
  <Explanation>
    El contexto ya se ha inicializado.  No se permiten actualizaciones en la configuración de la sesión.
  </Explanation>
  <UserResponse>
    Sólo puede modificar la configuración de la sesión antes de iniciar la aplicación.
  </UserResponse>
</Message>
<Message ID="SRVE0251W" severity="W" prefix="yes">
  <MsgText pgmKey="threadpool.not.used" varFormat="Java">
    threadpool.not.used=SRVE0251W: No se está utilizando la agrupación de hebras configurada en el contenedor web.
  </MsgText>
  <Explanation>
    Existe una configuración de agrupaciones de hebras en el contenedor web. Cualquier cambio que se realice en este elemento de configuración no redundará en cambios en el sistema.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE8039E" severity="E" prefix="yes">
  <MsgText pgmKey="not.called.from.within.the.context.of.a.dispatch.for.this.request" varFormat="Java">
    not.called.from.within.the.context.of.a.dispatch.for.this.request=SRVE8039E: El método startAsync no se ha llamado desde dentro del contexto de un envío de esta solicitud
  </MsgText>
  <Explanation>
    Al método startAsync se le debe llamar desde dentro del contexto de un contenedor tanto si se trata de la solicitud original como si se trata de un envío asíncrono subsiguiente.
  </Explanation>
  <UserResponse>
    Verifique que no haya problemas de lógica de código o de temporización que causen este comportamiento no permitido.
  </UserResponse>
</Message>
<Message ID="SRVE8112W" severity="W" prefix="yes">
  <MsgText pgmKey="programmatic.session.cookie.retrieve.cell.xml.failure" varFormat="Java">
    programmatic.session.cookie.retrieve.cell.xml.failure=SRVE8112W: Se ha producido un problema al recuperar los patrones de cookie restringida.  No se han establecido restricciones sobre la configuración de sesión programática.
  </MsgText>
  <Explanation>
    Se ha producido un problema al recuperar los patrones de cookie restringida de la configuración.
  </Explanation>
  <UserResponse>
    Verifique que cell.xml existe y que puede leerse.
  </UserResponse>
</Message>
<Message ID="SRVE8050E" severity="E" prefix="yes">
  <MsgText pgmKey="Place.servlet.class.on.classpath.of.the.application.server" varFormat="Java">
    Place.servlet.class.on.classpath.of.the.application.server=SRVE8050E: Coloque la clase de servlet en la vía de acceso de clases del servidor de aplicaciones {0}, {1}: {2}.
  </MsgText>
  <Explanation>
    Se han producido errores en la vía de acceso de clases al servir al servlet.
  </Explanation>
  <UserResponse>
    Coloque la clase de servlet en la vía de acceso de clases del servidor de aplicaciones.
  </UserResponse>
</Message>
<Message ID="SRVE0097I" severity="I" prefix="yes">
  <MsgText pgmKey="Servlet.unload.initiated:.{0}" varFormat="Java">
    Servlet.unload.initiated:.{0}=SRVE0097I: Se ha iniciado la descarga del servlet: {0}
  </MsgText>
  <Explanation>
    El servlet especificado está siendo descargado desde el servicio.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0289E" severity="E" prefix="yes">
  <MsgText pgmKey="failed.to.create.temp.directory" varFormat="Java">
    failed.to.create.temp.directory=SRVE0289E: No se ha podido crear el directorio temporal: {0}
  </MsgText>
  <Explanation>
    Es un error del sistema.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SRVE0212E" severity="E" prefix="yes">
  <MsgText pgmKey="class.{0}.not.accessible" varFormat="Java">
    class.{0}.not.accessible=SRVE0212E: La clase {0} no está accesible
  </MsgText>
  <Explanation>
    Excepción interna.
  </Explanation>
  <UserResponse>
    Si el problema persiste, puede haber información adicional disponible si busca el ID del mensaje en los sitios Web siguientes: página de soporte de IBM WebSphere Application Server: http://www.ibm.com/software/webservers/appserv/was/support/   página de soporte de IBM WebSphere Application Server for z/OS: http://www.ibm.com/software/webservers/appserv/zos_os390/support/
  </UserResponse>
</Message>
<Message ID="SRVE0234I" severity="I" prefix="yes">
  <MsgText pgmKey="Application.classpath" varFormat="Java">
    Application.classpath=SRVE0234I: Vía de acceso de clases de aplicación=[{0}]
  </MsgText>
  <Explanation>
    ninguna
  </Explanation>
  <UserResponse>
    ninguna
  </UserResponse>
</Message>
<Message ID="SRVE8031W" severity="W" prefix="yes">
  <MsgText pgmKey="exception.occurred.while.initializing.ServletContainerInitializers.class.lookup" varFormat="Java">
    exception.occurred.while.initializing.ServletContainerInitializers.class.lookup=SRVE8031W: Se ha producido un problema al buscar la clase, {0}, durante la exploración de ServletContainerInitializer HandlesTypes para {1}.
  </MsgText>
  <Explanation>
    La clase no se ha encontrado en el cargador de clases al procesar la anotación ServletContainerInitializer HandlesTypes.
  </Explanation>
  <UserResponse>
    Asegúrese de que exista la clase dentro del cargador de clases del módulo web.
  </UserResponse>
</Message>
<!-- END MESSAGES -->
</TMSSource>
